  INFO root [2019-09-21 09:28:05,308] - 
  INFO root [2019-09-21 09:28:05,309] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-21 09:28:05,310] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-21 09:28:05,310] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-21 09:28:05,310] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-21 09:28:05,322] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.26.195 ( 10.15.26.195:51535 )
  INFO root [2019-09-21 09:28:05,323] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.26.195:51535, SOMAXCONN=5.
  INFO root [2019-09-21 09:28:05,323] - server-configs:
  INFO root [2019-09-21 09:28:05,324] - 	internalTcpAddr : 10.15.26.195:51535
  INFO root [2019-09-21 09:28:05,324] - 	componentID : 4000
  WARN root [2019-09-21 09:28:05,324] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-21 09:28:06,829] - ---- dbmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: 11178A1C367680938C8E416B64844ACD. Config: Debug X86. Built: 09:55:16 May  8 2019. AppID: 4000. UID: 41181. PID: 11028 ----
  INFO root [2019-09-21 09:28:06,830] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-21 09:28:06,830] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-21 09:28:06,831] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-21 09:28:07,037] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [2019-09-21 09:28:07,038] - Script::install(): KBExtra is successfully!
  INFO root [2019-09-21 09:28:07,295] - Dbmgr::initializeEnd: digest(6C1130BA93B38570A1A313CCA3BEAA21)
S_INFO root [2019-09-21 09:28:07,295] - onDBMgrReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2019-09-21 09:28:07,296] - TelnetServer server is running on port 32000
  INFO root [2019-09-21 09:28:07,296] - Dbmgr::initInterfacesHandler: interfaces addr(127.0.0.1:30099), accountType:(interfaces), chargeType:(interfaces).
  WARN root [2019-09-21 09:28:07,558] - DBUtil::initialize: db(default) password is not encrypted!
please use password(rsa):
43C4DF850E37C9BF8B2ABDD2316C82C2167517EF709F73302552CBDCE6EB2BF76E095560BC10BAE04BA53FDAFB72D19089D246700DFBB374B8B53DA6FDDF59333CEF8B07CB02B8967899DBFAEE3AD81A129FDC650D1DADD6BE290EF780B20190212D532A1B1E6815DA12AEBC4D9240D677197927BBF068098FCEFE709D6BCC22
 DEBUG root [2019-09-21 09:28:07,562] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 09:28:07,771] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
  INFO root [2019-09-21 09:28:07,771] - DBUtil::createInterface[0x1afee0c]: interface=default, dbtype=mysql, ip=localhost, port=3306, currdatabase=kbe, username=kbe, connected=yes.

  INFO root [2019-09-21 09:28:07,771] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-21 09:28:07,772] - ThreadPool::createThreadPool: successfully(5), newThreadCount=5, normalMaxThreadCount=5, maxThreadCount=5
 DEBUG root [2019-09-21 09:28:07,791] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 09:28:07,795] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 09:28:07,796] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 09:28:07,797] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 09:28:07,816] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 09:28:07,947] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 09:28:07,947] - DBThread::onStart(): 0x1ef80d8!
  INFO root [2019-09-21 09:28:07,953] - DBInterfaceMysql::query: successfully!
 DEBUG root [2019-09-21 09:28:08,047] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 09:28:08,047] - DBThread::onStart(): 0x1ef8380!
 DEBUG root [2019-09-21 09:28:08,112] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 09:28:08,112] - DBThread::onStart(): 0x1ef8160!
  INFO root [2019-09-21 09:28:08,145] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 09:28:08,162] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-21 09:28:08,163] - DBTask::process(): took 0.21 seconds
sql:(CREATE TABLE IF NOT EXISTS kbe_accountinfos (`accountName` varchar(128) not null, PRIMARY KEY idKey (`accountName`),`password` varchar(255) not null,`bindata` blob,`email` varchar(191) not null, UNIQUE KEY `email` (`email`),`entityDBID` bigint(20) unsigned not null DEFAULT 0, UNIQUE KEY `entityDBID` (`entityDBID`),`flags` int unsigned not null DEFAULT 0,`deadline` bigint(20) not null DEFAULT 0,`regtime` bigint(20) not null DEFAULT 0,`lasttime` bigint(20) not null DEFAULT 0,`numlogin` int unsigned not null DEFAULT 0)ENGINE=InnoDB)
 DEBUG root [2019-09-21 09:28:08,213] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 09:28:08,213] - DBThread::onStart(): 0x1ef8958!
 DEBUG root [2019-09-21 09:28:08,255] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 09:28:08,255] - DBThread::onStart(): 0x1ef8628!
  WARN root [2019-09-21 09:28:08,346] - DBTask::process(): took 0.23 seconds
sql:(show index from tbl_Account)
  INFO root [2019-09-21 09:28:08,397] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 09:28:08,397] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 09:28:08,398] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 09:28:08,398] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 09:28:08,398] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-21 09:28:08,400] - DBTask::process(): took 0.24 seconds
sql:(show index from tbl_Room)
  INFO root [2019-09-21 09:28:08,437] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-21 09:28:08,438] - DBTask::process(): took 0.39 seconds
sql:(select distinct(serverGroupID) from kbe_entitylog)
  WARN root [2019-09-21 09:28:08,530] - DBTask::process(): took 0.27 seconds
sql:(insert into kbe_serverlog (heartbeatTime, serverGroupID) values(1569029288,41181) ON DUPLICATE KEY UPDATE heartbeatTime=1569029288)
  WARN root [2019-09-21 09:28:08,556] - d:\kbengine\kbe\src\server\dbmgr\dbmgr.cpp:306: Profile scriptCall took 1.26 seconds
  INFO root [2019-09-21 09:28:09,009] - ---- dbmgr is running ----
 DEBUG root [2019-09-21 09:28:09,063] - Components::process(): Request for the process of identity...
  INFO root [2019-09-21 09:28:15,175] - ServerApp::onRegisterNewApp: uid:41181, username:error_name, componentType:interfaces, componentID:3000, globalorderID=-1, grouporderID=-1, intaddr:10.15.26.195, intport:30099, extaddr:nonsupport, extport:0,  from 127.0.0.1:30099/0/0/0.
  INFO root [2019-09-21 09:28:15,176] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-21 09:28:15,176] - Components::addComponent[interfaces], uid=41181, componentID=3000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-21 09:28:16,195] - Components::findComponents: find logger(1)...
 ERROR root [2019-09-21 09:28:17,697] - BundleBroadcast::receive: failed! It can be caused by the firewall, the broadcastaddr, etc.Maybe broadcastaddr is not a LAN ADDR, or the Machine process is not running.
 ERROR root [2019-09-21 09:28:17,698] - Components::findComponents: receive error!
  WARN root [2019-09-21 09:28:17,699] - Components::findComponents: not found logger!
  INFO root [2019-09-21 09:28:18,736] - Components::process(): Found all the components!
  INFO root [2019-09-21 09:29:34,220] - 
  INFO root [2019-09-21 09:29:34,221] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-21 09:29:34,222] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-21 09:29:34,223] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-21 09:29:34,223] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-21 09:29:34,232] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.26.195 ( 10.15.26.195:51562 )
  INFO root [2019-09-21 09:29:34,232] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.26.195:51562, SOMAXCONN=5.
  INFO root [2019-09-21 09:29:34,233] - server-configs:
  INFO root [2019-09-21 09:29:34,233] - 	internalTcpAddr : 10.15.26.195:51562
  INFO root [2019-09-21 09:29:34,233] - 	componentID : 4000
  WARN root [2019-09-21 09:29:34,234] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-21 09:29:35,740] - ---- dbmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: 11178A1C367680938C8E416B64844ACD. Config: Debug X86. Built: 09:55:16 May  8 2019. AppID: 4000. UID: 41181. PID: 8504 ----
  INFO root [2019-09-21 09:29:35,741] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-21 09:29:35,741] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-21 09:29:35,742] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-21 09:29:35,934] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [2019-09-21 09:29:35,934] - Script::install(): KBExtra is successfully!
  INFO root [2019-09-21 09:29:36,058] - Dbmgr::initializeEnd: digest(6C1130BA93B38570A1A313CCA3BEAA21)
S_INFO root [2019-09-21 09:29:36,059] - onDBMgrReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2019-09-21 09:29:36,059] - TelnetServer server is running on port 32000
  INFO root [2019-09-21 09:29:36,059] - Dbmgr::initInterfacesHandler: interfaces addr(127.0.0.1:30099), accountType:(interfaces), chargeType:(interfaces).
  WARN root [2019-09-21 09:29:36,274] - DBUtil::initialize: db(default) password is not encrypted!
please use password(rsa):
BEDC0B8705BCB6105D1CC2659ABF4FD8A2BFBE9BCF86E6B33D1E8089E0711326DE1B8F09F387234160894EED8FE6BB1D7FA8B2F21C567771348072D9C51FB5A503A1DFAB734F2EF838FDB918D1C3D1FB5F7D231D6381D439F8207FFB33093627BD5C47FCE0FC394D1B53002AE8B6B044B9DAEB28B5C4C588AACFA6C33CEB559B
 DEBUG root [2019-09-21 09:29:36,278] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 09:29:36,405] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
  INFO root [2019-09-21 09:29:36,405] - DBUtil::createInterface[0x3bf2a8]: interface=default, dbtype=mysql, ip=localhost, port=3306, currdatabase=kbe, username=kbe, connected=yes.

  INFO root [2019-09-21 09:29:36,405] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-21 09:29:36,406] - ThreadPool::createThreadPool: successfully(5), newThreadCount=5, normalMaxThreadCount=5, maxThreadCount=5
 DEBUG root [2019-09-21 09:29:36,406] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 09:29:36,407] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 09:29:36,407] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 09:29:36,407] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 09:29:36,423] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 09:29:36,464] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 09:29:36,464] - DBThread::onStart(): 0x18802c0!
 DEBUG root [2019-09-21 09:29:36,530] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 09:29:36,530] - DBThread::onStart(): 0x18803d0!
 DEBUG root [2019-09-21 09:29:36,564] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 09:29:36,564] - DBThread::onStart(): 0x1880348!
 DEBUG root [2019-09-21 09:29:36,622] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 09:29:36,623] - DBThread::onStart(): 0x1880458!
 DEBUG root [2019-09-21 09:29:36,689] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 09:29:36,689] - DBThread::onStart(): 0x1880cd8!
  INFO root [2019-09-21 09:29:36,727] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 09:29:36,747] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 09:29:36,748] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 09:29:36,748] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 09:29:36,749] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 09:29:36,764] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 09:29:36,780] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 09:29:36,780] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 09:29:36,781] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-21 09:29:36,871] - d:\kbengine\kbe\src\server\dbmgr\dbmgr.cpp:306: Profile scriptCall took 0.81 seconds
  INFO root [2019-09-21 09:29:37,283] - ---- dbmgr is running ----
 DEBUG root [2019-09-21 09:29:37,283] - Components::process(): Request for the process of identity...
  INFO root [2019-09-21 09:29:43,400] - ServerApp::onRegisterNewApp: uid:41181, username:error_name, componentType:interfaces, componentID:3000, globalorderID=-1, grouporderID=-1, intaddr:10.15.26.195, intport:30099, extaddr:nonsupport, extport:0,  from 127.0.0.1:30099/0/0/0.
  INFO root [2019-09-21 09:29:43,401] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-21 09:29:43,401] - Components::addComponent[interfaces], uid=41181, componentID=3000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-21 09:29:44,417] - Components::findComponents: find logger(1)...
  INFO root [2019-09-21 09:29:44,484] - Components::findComponents: found logger, addr:10.15.26.195:51565
  INFO root [2019-09-21 09:29:44,484] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-21 09:29:44,484] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  WARN root [2019-09-21 09:29:44,485] - Forwarding logs to logger[10.15.26.195:51565/0/0/0]...
  INFO root [2019-09-21 09:39:31,642] - EventDispatcher()::~EventDispatcher: Num timers = 2
  INFO root [2019-09-21 09:40:36,978] - 
  INFO root [2019-09-21 09:40:36,979] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-21 09:40:36,980] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-21 09:40:36,980] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-21 09:40:36,980] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-21 09:40:36,991] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.26.195 ( 10.15.26.195:51759 )
  INFO root [2019-09-21 09:40:36,991] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.26.195:51759, SOMAXCONN=5.
  INFO root [2019-09-21 09:40:36,991] - server-configs:
  INFO root [2019-09-21 09:40:36,991] - 	internalTcpAddr : 10.15.26.195:51759
  INFO root [2019-09-21 09:40:36,992] - 	componentID : 4000
  WARN root [2019-09-21 09:40:36,995] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  WARN root [2019-09-21 09:40:37,337] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20466, errorno=10048
  INFO root [2019-09-21 09:40:37,341] - ---- dbmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: 11178A1C367680938C8E416B64844ACD. Config: Debug X86. Built: 09:55:16 May  8 2019. AppID: 4000. UID: 41181. PID: 14128 ----
  INFO root [2019-09-21 09:40:37,342] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-21 09:40:37,342] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-21 09:40:37,343] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-21 09:40:37,567] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [2019-09-21 09:40:37,567] - Script::install(): KBExtra is successfully!
  INFO root [2019-09-21 09:40:37,720] - Dbmgr::initializeEnd: digest(6C1130BA93B38570A1A313CCA3BEAA21)
S_INFO root [2019-09-21 09:40:37,721] - onDBMgrReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2019-09-21 09:40:37,721] - TelnetServer server is running on port 32000
  INFO root [2019-09-21 09:40:37,722] - Dbmgr::initInterfacesHandler: interfaces addr(127.0.0.1:30099), accountType:(interfaces), chargeType:(interfaces).
  WARN root [2019-09-21 09:40:37,944] - DBUtil::initialize: db(default) password is not encrypted!
please use password(rsa):
C99A7B9EF1EEDC37F862284BDB475B9CCD0FCCB8E993126B79D8AB6C2282E204420BA6A4574858E30DB27F012CDFE3F13A0EB612C8FAAFCBC8D776D20C3BF9B8F054314181C488B37FD33C30481D3153DE4A69F68F5B6FCC74B2C62D2E165B21B276341B240C93EA644A064659251570B8E44F26DD6B1C4813FA0096D4CDCB84
 DEBUG root [2019-09-21 09:40:37,948] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 09:40:38,054] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
  INFO root [2019-09-21 09:40:38,054] - DBUtil::createInterface[0x57f0c8]: interface=default, dbtype=mysql, ip=localhost, port=3306, currdatabase=kbe, username=kbe, connected=yes.

  INFO root [2019-09-21 09:40:38,054] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-21 09:40:38,055] - ThreadPool::createThreadPool: successfully(5), newThreadCount=5, normalMaxThreadCount=5, maxThreadCount=5
 DEBUG root [2019-09-21 09:40:38,058] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 09:40:38,059] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 09:40:38,068] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 09:40:38,074] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 09:40:38,074] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 09:40:38,186] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 09:40:38,186] - DBThread::onStart(): 0x17ae910!
 DEBUG root [2019-09-21 09:40:38,309] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 09:40:38,309] - DBThread::onStart(): 0x17ae558!
  INFO root [2019-09-21 09:40:38,312] - DBInterfaceMysql::query: successfully!
 DEBUG root [2019-09-21 09:40:38,451] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 09:40:38,451] - DBThread::onStart(): 0x17ae998!
 DEBUG root [2019-09-21 09:40:38,510] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 09:40:38,510] - DBThread::onStart(): 0x17ae800!
 DEBUG root [2019-09-21 09:40:38,576] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 09:40:38,576] - DBThread::onStart(): 0x17ae668!
  INFO root [2019-09-21 09:40:38,642] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 09:40:38,646] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-21 09:40:38,647] - DBTask::process(): took 0.33 seconds
sql:(select distinct(serverGroupID) from kbe_entitylog)
  INFO root [2019-09-21 09:40:38,672] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 09:40:38,672] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 09:40:38,673] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-21 09:40:38,673] - DBTask::process(): took 0.36 seconds
sql:(CREATE TABLE IF NOT EXISTS kbe_accountinfos (`accountName` varchar(128) not null, PRIMARY KEY idKey (`accountName`),`password` varchar(255) not null,`bindata` blob,`email` varchar(191) not null, UNIQUE KEY `email` (`email`),`entityDBID` bigint(20) unsigned not null DEFAULT 0, UNIQUE KEY `entityDBID` (`entityDBID`),`flags` int unsigned not null DEFAULT 0,`deadline` bigint(20) not null DEFAULT 0,`regtime` bigint(20) not null DEFAULT 0,`lasttime` bigint(20) not null DEFAULT 0,`numlogin` int unsigned not null DEFAULT 0)ENGINE=InnoDB)
  INFO root [2019-09-21 09:40:38,678] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 09:40:38,679] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 09:40:38,679] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-21 09:40:38,763] - DBTask::process(): took 0.31 seconds
sql:(insert into kbe_serverlog (heartbeatTime, serverGroupID) values(1569030038,41181) ON DUPLICATE KEY UPDATE heartbeatTime=1569030038)
  WARN root [2019-09-21 09:40:38,785] - d:\kbengine\kbe\src\server\dbmgr\dbmgr.cpp:306: Profile scriptCall took 1.06 seconds
  INFO root [2019-09-21 09:40:39,241] - ---- dbmgr is running ----
 DEBUG root [2019-09-21 09:40:39,241] - Components::process(): Request for the process of identity...
  INFO root [2019-09-21 09:40:45,379] - ServerApp::onRegisterNewApp: uid:41181, username:error_name, componentType:interfaces, componentID:3000, globalorderID=-1, grouporderID=-1, intaddr:10.15.26.195, intport:30099, extaddr:nonsupport, extport:0,  from 127.0.0.1:30099/0/0/0.
  INFO root [2019-09-21 09:40:45,380] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-21 09:40:45,380] - Components::addComponent[interfaces], uid=41181, componentID=3000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-21 09:40:46,400] - Components::findComponents: find logger(1)...
  INFO root [2019-09-21 09:40:46,405] - Components::findComponents: found logger, addr:10.15.26.195:51754
  INFO root [2019-09-21 09:40:46,405] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-21 09:40:46,406] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-21 09:40:46,440] - Components::process(): Found all the components!
  WARN root [2019-09-21 09:40:46,481] - Forwarding logs to logger[10.15.26.195:51754/0/0/0]...
 ERROR root [2019-09-21 09:45:33,916] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=467633622819, lastReceivedTime=467362714930, inactivityExceptionPeriod=599440722)
  INFO root [2019-09-21 09:45:33,918] - The following logs sent to logger failed:
  WARN root [2019-09-21 09:45:33,918] - ==> [2019-09-21 09:45:33 914] TCPPacketReceiver::processPendingEvents(10.15.26.195:51754): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-21 09:45:33,918] - ==> [2019-09-21 09:45:33 916] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=467633622819, lastReceivedTime=467362714930, inactivityExceptionPeriod=599440722)
  INFO root [2019-09-21 09:48:09,603] - 
  INFO root [2019-09-21 09:48:09,604] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-21 09:48:09,606] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-21 09:48:09,606] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-21 09:48:09,606] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-21 09:48:09,621] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.26.195 ( 10.15.26.195:51846 )
  INFO root [2019-09-21 09:48:09,621] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.26.195:51846, SOMAXCONN=5.
  INFO root [2019-09-21 09:48:09,621] - server-configs:
  INFO root [2019-09-21 09:48:09,621] - 	internalTcpAddr : 10.15.26.195:51846
  INFO root [2019-09-21 09:48:09,621] - 	componentID : 4000
  WARN root [2019-09-21 09:48:09,625] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-21 09:48:11,129] - ---- dbmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: 11178A1C367680938C8E416B64844ACD. Config: Debug X86. Built: 09:55:16 May  8 2019. AppID: 4000. UID: 41181. PID: 4868 ----
  INFO root [2019-09-21 09:48:11,130] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-21 09:48:11,130] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-21 09:48:11,131] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-21 09:48:11,333] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [2019-09-21 09:48:11,333] - Script::install(): KBExtra is successfully!
  INFO root [2019-09-21 09:48:11,679] - Dbmgr::initializeEnd: digest(08E2C345060C0E06DB568503A5EFB1A8)
S_INFO root [2019-09-21 09:48:11,679] - onDBMgrReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2019-09-21 09:48:11,680] - TelnetServer server is running on port 32000
  INFO root [2019-09-21 09:48:11,717] - Dbmgr::initInterfacesHandler: interfaces addr(127.0.0.1:30099), accountType:(interfaces), chargeType:(interfaces).
  WARN root [2019-09-21 09:48:11,967] - DBUtil::initialize: db(default) password is not encrypted!
please use password(rsa):
167BB35A365A6C314CC64860C31CA88A38A1824F663A0DF3447CD913D8CFF0C2D221EA493DADF5E07BDEAB52339799E0AA2F24F89C37EBC4589221438547D0F603A71361DC2F45A750DF2BFB85FE639B05298F2645A6CF79741340B576CF24A56ECF1AD4847A6F7CFC5AC59A144A4A14493896FB43C94B73867BE1A75CA9B70F
 DEBUG root [2019-09-21 09:48:11,975] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 09:48:12,106] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
  INFO root [2019-09-21 09:48:12,106] - DBUtil::createInterface[0x1aff058]: interface=default, dbtype=mysql, ip=localhost, port=3306, currdatabase=kbe, username=kbe, connected=yes.

  INFO root [2019-09-21 09:48:12,107] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-21 09:48:12,107] - ThreadPool::createThreadPool: successfully(5), newThreadCount=5, normalMaxThreadCount=5, maxThreadCount=5
 DEBUG root [2019-09-21 09:48:12,107] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 09:48:12,107] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 09:48:12,108] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 09:48:12,108] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 09:48:12,124] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 09:48:12,283] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 09:48:12,283] - DBThread::onStart(): 0x1b017f8!
 DEBUG root [2019-09-21 09:48:12,411] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 09:48:12,411] - DBThread::onStart(): 0x1b00c48!
  INFO root [2019-09-21 09:48:12,413] - DBInterfaceMysql::query: successfully!
 DEBUG root [2019-09-21 09:48:12,504] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 09:48:12,504] - DBThread::onStart(): 0x1b01198!
  INFO root [2019-09-21 09:48:12,537] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 09:48:12,554] - DBInterfaceMysql::query: successfully!
 DEBUG root [2019-09-21 09:48:12,596] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 09:48:12,596] - DBThread::onStart(): 0x1b00e68!
 DEBUG root [2019-09-21 09:48:12,638] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 09:48:12,638] - DBThread::onStart(): 0x1b00de0!
  INFO root [2019-09-21 09:48:12,687] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 09:48:12,714] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 09:48:12,714] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 09:48:12,715] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 09:48:12,715] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 09:48:12,715] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-21 09:48:12,716] - DBTask::process(): took 0.30 seconds
sql:(select distinct(serverGroupID) from kbe_entitylog)
  WARN root [2019-09-21 09:48:12,770] - DBTask::process(): took 0.27 seconds
sql:(insert into kbe_serverlog (heartbeatTime, serverGroupID) values(1569030492,41181) ON DUPLICATE KEY UPDATE heartbeatTime=1569030492)
  WARN root [2019-09-21 09:48:12,854] - d:\kbengine\kbe\src\server\dbmgr\dbmgr.cpp:306: Profile scriptCall took 1.18 seconds
  INFO root [2019-09-21 09:48:13,331] - ---- dbmgr is running ----
 DEBUG root [2019-09-21 09:48:13,332] - Components::process(): Request for the process of identity...
  INFO root [2019-09-21 09:48:19,459] - ServerApp::onRegisterNewApp: uid:41181, username:error_name, componentType:interfaces, componentID:3000, globalorderID=-1, grouporderID=-1, intaddr:10.15.26.195, intport:30099, extaddr:nonsupport, extport:0,  from 127.0.0.1:30099/0/0/0.
  INFO root [2019-09-21 09:48:19,459] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-21 09:48:19,459] - Components::addComponent[interfaces], uid=41181, componentID=3000, globalorderid=1, grouporderid=1, totalcount=1
  WARN root [2019-09-21 09:48:19,461] - TCPPacketReceiver::processPendingEvents(10.15.26.195:51859): Throwing REASON_GENERAL_NETWORK - WSAECONNABORTED
  INFO root [2019-09-21 09:48:20,479] - Components::findComponents: find logger(1)...
  INFO root [2019-09-21 09:48:20,483] - Components::findComponents: found logger, addr:10.15.26.195:51845
  INFO root [2019-09-21 09:48:20,483] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-21 09:48:20,483] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-21 09:48:20,499] - Components::process(): Found all the components!
  WARN root [2019-09-21 09:48:20,508] - Forwarding logs to logger[10.15.26.195:51845/0/0/0]...
 ERROR root [2019-09-21 09:52:20,731] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=471701777545, lastReceivedTime=471590628226, inactivityExceptionPeriod=599468219)
  INFO root [2019-09-21 09:52:20,736] - The following logs sent to logger failed:
  WARN root [2019-09-21 09:52:20,736] - ==> [2019-09-21 09:52:20 730] TCPPacketReceiver::processPendingEvents(10.15.26.195:51845): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-21 09:52:20,736] - ==> [2019-09-21 09:52:20 732] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=471701777545, lastReceivedTime=471590628226, inactivityExceptionPeriod=599468219)
  INFO root [2019-09-21 09:54:15,527] - 
  INFO root [2019-09-21 09:54:15,539] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-21 09:54:15,540] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-21 09:54:15,540] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-21 09:54:15,540] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-21 09:54:15,575] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.26.195 ( 10.15.26.195:51947 )
  INFO root [2019-09-21 09:54:15,576] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.26.195:51947, SOMAXCONN=5.
  INFO root [2019-09-21 09:54:15,576] - server-configs:
  INFO root [2019-09-21 09:54:15,576] - 	internalTcpAddr : 10.15.26.195:51947
  INFO root [2019-09-21 09:54:15,576] - 	componentID : 4000
  WARN root [2019-09-21 09:54:15,587] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-21 09:54:17,113] - ---- dbmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: 11178A1C367680938C8E416B64844ACD. Config: Debug X86. Built: 09:55:16 May  8 2019. AppID: 4000. UID: 41181. PID: 12640 ----
  INFO root [2019-09-21 09:54:17,114] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-21 09:54:17,114] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-21 09:54:17,114] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-21 09:54:17,312] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [2019-09-21 09:54:17,312] - Script::install(): KBExtra is successfully!
  INFO root [2019-09-21 09:54:17,452] - Dbmgr::initializeEnd: digest(6C1130BA93B38570A1A313CCA3BEAA21)
S_INFO root [2019-09-21 09:54:17,453] - onDBMgrReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2019-09-21 09:54:17,453] - TelnetServer server is running on port 32000
  INFO root [2019-09-21 09:54:17,454] - Dbmgr::initInterfacesHandler: interfaces addr(127.0.0.1:30099), accountType:(interfaces), chargeType:(interfaces).
  WARN root [2019-09-21 09:54:17,675] - DBUtil::initialize: db(default) password is not encrypted!
please use password(rsa):
82CC745A8C29E1586857064E8FCA1D0B491C77E7BEAFF0C6F0A70A83C529E0831AB963335406035DFFB3F5CCB9085BF51D5ABD7831A695A6F9BC5CC642933A01CA41B9B6B32AC178D51527DFC8DD7E6C07B2892B2434372A3D2C148A0370C1F4E5AAF1ACB0DF7C839853519C1BC08E7E76202850472F471010BE898A0BB614D4
 DEBUG root [2019-09-21 09:54:17,679] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 09:54:17,827] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
  INFO root [2019-09-21 09:54:17,828] - DBUtil::createInterface[0x1aff248]: interface=default, dbtype=mysql, ip=localhost, port=3306, currdatabase=kbe, username=kbe, connected=yes.

  INFO root [2019-09-21 09:54:17,828] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-21 09:54:17,829] - ThreadPool::createThreadPool: successfully(5), newThreadCount=5, normalMaxThreadCount=5, maxThreadCount=5
 DEBUG root [2019-09-21 09:54:17,830] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 09:54:17,833] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 09:54:17,839] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 09:54:17,840] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 09:54:17,848] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 09:54:17,926] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 09:54:17,926] - DBThread::onStart(): 0x1bbdbc0!
 DEBUG root [2019-09-21 09:54:17,989] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 09:54:17,989] - DBThread::onStart(): 0x1bbdc48!
 DEBUG root [2019-09-21 09:54:18,120] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 09:54:18,121] - DBThread::onStart(): 0x1bbe110!
  INFO root [2019-09-21 09:54:18,122] - DBInterfaceMysql::query: successfully!
 DEBUG root [2019-09-21 09:54:18,198] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 09:54:18,198] - DBThread::onStart(): 0x1bbe198!
 DEBUG root [2019-09-21 09:54:18,281] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 09:54:18,281] - DBThread::onStart(): 0x1bbe220!
  INFO root [2019-09-21 09:54:18,331] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 09:54:18,356] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 09:54:18,356] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 09:54:18,356] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-21 09:54:18,357] - DBTask::process(): took 0.23 seconds
sql:(CREATE TABLE IF NOT EXISTS kbe_accountinfos (`accountName` varchar(128) not null, PRIMARY KEY idKey (`accountName`),`password` varchar(255) not null,`bindata` blob,`email` varchar(191) not null, UNIQUE KEY `email` (`email`),`entityDBID` bigint(20) unsigned not null DEFAULT 0, UNIQUE KEY `entityDBID` (`entityDBID`),`flags` int unsigned not null DEFAULT 0,`deadline` bigint(20) not null DEFAULT 0,`regtime` bigint(20) not null DEFAULT 0,`lasttime` bigint(20) not null DEFAULT 0,`numlogin` int unsigned not null DEFAULT 0)ENGINE=InnoDB)
  INFO root [2019-09-21 09:54:18,357] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 09:54:18,358] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 09:54:18,358] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-21 09:54:18,490] - DBTask::process(): took 0.37 seconds
sql:(insert into kbe_serverlog (heartbeatTime, serverGroupID) values(1569030858,41181) ON DUPLICATE KEY UPDATE heartbeatTime=1569030858)
  INFO root [2019-09-21 09:54:18,523] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-21 09:54:18,525] - DBTask::process(): took 0.40 seconds
sql:(select distinct(serverGroupID) from kbe_entitylog)
  WARN root [2019-09-21 09:54:18,550] - d:\kbengine\kbe\src\server\dbmgr\dbmgr.cpp:306: Profile scriptCall took 1.10 seconds
  INFO root [2019-09-21 09:54:18,997] - ---- dbmgr is running ----
 DEBUG root [2019-09-21 09:54:18,997] - Components::process(): Request for the process of identity...
  INFO root [2019-09-21 09:54:25,129] - ServerApp::onRegisterNewApp: uid:41181, username:error_name, componentType:interfaces, componentID:3000, globalorderID=-1, grouporderID=-1, intaddr:10.15.26.195, intport:30099, extaddr:nonsupport, extport:0,  from 127.0.0.1:30099/0/0/0.
  INFO root [2019-09-21 09:54:25,130] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-21 09:54:25,130] - Components::addComponent[interfaces], uid=41181, componentID=3000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-21 09:54:26,133] - Components::findComponents: find logger(1)...
  INFO root [2019-09-21 09:54:26,183] - Components::findComponents: found logger, addr:10.15.26.195:51946
  INFO root [2019-09-21 09:54:26,183] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-21 09:54:26,184] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  WARN root [2019-09-21 09:54:26,185] - Forwarding logs to logger[10.15.26.195:51946/0/0/0]...
 ERROR root [2019-09-21 09:56:00,285] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=473897324951, lastReceivedTime=473749881539, inactivityExceptionPeriod=599493286)
  INFO root [2019-09-21 09:56:00,295] - The following logs sent to logger failed:
  WARN root [2019-09-21 09:56:00,295] - ==> [2019-09-21 09:56:00 285] TCPPacketReceiver::processPendingEvents(10.15.26.195:51946): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-21 09:56:00,296] - ==> [2019-09-21 09:56:00 287] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=473897324951, lastReceivedTime=473749881539, inactivityExceptionPeriod=599493286)
  INFO root [2019-09-21 09:56:02,001] - 
  INFO root [2019-09-21 09:56:02,002] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-21 09:56:02,003] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-21 09:56:02,004] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-21 09:56:02,004] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-21 09:56:02,015] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.26.195 ( 10.15.26.195:52013 )
  INFO root [2019-09-21 09:56:02,015] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.26.195:52013, SOMAXCONN=5.
  INFO root [2019-09-21 09:56:02,015] - server-configs:
  INFO root [2019-09-21 09:56:02,015] - 	internalTcpAddr : 10.15.26.195:52013
  INFO root [2019-09-21 09:56:02,015] - 	componentID : 4000
  WARN root [2019-09-21 09:56:02,016] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-21 09:56:03,521] - ---- dbmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: 11178A1C367680938C8E416B64844ACD. Config: Debug X86. Built: 09:55:16 May  8 2019. AppID: 4000. UID: 41181. PID: 1852 ----
  INFO root [2019-09-21 09:56:03,522] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-21 09:56:03,522] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-21 09:56:03,522] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-21 09:56:03,725] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [2019-09-21 09:56:03,726] - Script::install(): KBExtra is successfully!
  INFO root [2019-09-21 09:56:03,858] - Dbmgr::initializeEnd: digest(6C1130BA93B38570A1A313CCA3BEAA21)
S_INFO root [2019-09-21 09:56:03,858] - onDBMgrReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2019-09-21 09:56:03,858] - TelnetServer server is running on port 32000
  INFO root [2019-09-21 09:56:03,859] - Dbmgr::initInterfacesHandler: interfaces addr(127.0.0.1:30099), accountType:(interfaces), chargeType:(interfaces).
  WARN root [2019-09-21 09:56:04,065] - DBUtil::initialize: db(default) password is not encrypted!
please use password(rsa):
16658AE268EFE998D265BDAE7384865A989932A048CBD99E6BD62FE32DF089B1FACC98470C8000E036D03E2780E582EB7CD6FF28D2F3602EB743862D546C276B6D5B7C0C7C93C072A4E6883A46EF488D3308A09C1ABED8B639318616F29708BD5B60BA05C2BFB764353C3BF28A0B3CE647CC48AF56503D79E514384E90A14CA5
 DEBUG root [2019-09-21 09:56:04,069] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 09:56:04,186] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
  INFO root [2019-09-21 09:56:04,187] - DBUtil::createInterface[0x1aff144]: interface=default, dbtype=mysql, ip=localhost, port=3306, currdatabase=kbe, username=kbe, connected=yes.

  INFO root [2019-09-21 09:56:04,187] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-21 09:56:04,188] - ThreadPool::createThreadPool: successfully(5), newThreadCount=5, normalMaxThreadCount=5, maxThreadCount=5
 DEBUG root [2019-09-21 09:56:04,188] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 09:56:04,188] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 09:56:04,188] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 09:56:04,188] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 09:56:04,190] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 09:56:04,313] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 09:56:04,313] - DBThread::onStart(): 0x1b8f740!
 DEBUG root [2019-09-21 09:56:04,373] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 09:56:04,374] - DBThread::onStart(): 0x1b8f850!
 DEBUG root [2019-09-21 09:56:04,449] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 09:56:04,449] - DBThread::onStart(): 0x1b8f9e8!
 DEBUG root [2019-09-21 09:56:04,499] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 09:56:04,499] - DBThread::onStart(): 0x1b8fa70!
  INFO root [2019-09-21 09:56:04,570] - DBInterfaceMysql::query: successfully!
 DEBUG root [2019-09-21 09:56:04,624] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 09:56:04,624] - DBThread::onStart(): 0x1b8f8d8!
  INFO root [2019-09-21 09:56:04,689] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 09:56:04,690] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 09:56:04,732] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 09:56:04,732] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 09:56:04,743] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 09:56:04,743] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 09:56:04,744] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-21 09:56:04,824] - DBTask::process(): took 0.25 seconds
sql:(show index from tbl_Account)
  WARN root [2019-09-21 09:56:04,875] - DBTask::process(): took 0.30 seconds
sql:(insert into kbe_serverlog (heartbeatTime, serverGroupID) values(1569030964,41181) ON DUPLICATE KEY UPDATE heartbeatTime=1569030964)
  INFO root [2019-09-21 09:56:04,943] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-21 09:56:04,943] - DBTask::process(): took 0.37 seconds
sql:(select distinct(serverGroupID) from kbe_entitylog)
  WARN root [2019-09-21 09:56:04,966] - d:\kbengine\kbe\src\server\dbmgr\dbmgr.cpp:306: Profile scriptCall took 1.11 seconds
  INFO root [2019-09-21 09:56:05,398] - ---- dbmgr is running ----
 DEBUG root [2019-09-21 09:56:05,398] - Components::process(): Request for the process of identity...
  INFO root [2019-09-21 09:56:11,521] - ServerApp::onRegisterNewApp: uid:41181, username:error_name, componentType:interfaces, componentID:3000, globalorderID=-1, grouporderID=-1, intaddr:10.15.26.195, intport:30099, extaddr:nonsupport, extport:0,  from 127.0.0.1:30099/0/0/0.
  INFO root [2019-09-21 09:56:11,521] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-21 09:56:11,522] - Components::addComponent[interfaces], uid=41181, componentID=3000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-21 09:56:12,538] - Components::findComponents: find logger(1)...
  INFO root [2019-09-21 09:56:12,639] - Components::findComponents: found logger, addr:10.15.26.195:52015
  INFO root [2019-09-21 09:56:12,639] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-21 09:56:12,639] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  WARN root [2019-09-21 09:56:12,640] - Forwarding logs to logger[10.15.26.195:52015/0/0/0]...
 ERROR root [2019-09-21 09:57:59,389] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=475088353224, lastReceivedTime=474815475737, inactivityExceptionPeriod=599494117)
  INFO root [2019-09-21 09:57:59,394] - The following logs sent to logger failed:
  WARN root [2019-09-21 09:57:59,395] - ==> [2019-09-21 09:57:59 387] TCPPacketReceiver::processPendingEvents(10.15.26.195:52015): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-21 09:57:59,395] - ==> [2019-09-21 09:57:59 389] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=475088353224, lastReceivedTime=474815475737, inactivityExceptionPeriod=599494117)
  INFO root [2019-09-21 09:58:01,119] - 
  INFO root [2019-09-21 09:58:01,123] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-21 09:58:01,124] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-21 09:58:01,124] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-21 09:58:01,125] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-21 09:58:01,148] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.26.195 ( 10.15.26.195:52070 )
  INFO root [2019-09-21 09:58:01,149] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.26.195:52070, SOMAXCONN=5.
  INFO root [2019-09-21 09:58:01,149] - server-configs:
  INFO root [2019-09-21 09:58:01,149] - 	internalTcpAddr : 10.15.26.195:52070
  INFO root [2019-09-21 09:58:01,149] - 	componentID : 4000
  WARN root [2019-09-21 09:58:01,154] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-21 09:58:02,677] - ---- dbmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: 11178A1C367680938C8E416B64844ACD. Config: Debug X86. Built: 09:55:16 May  8 2019. AppID: 4000. UID: 41181. PID: 13828 ----
  INFO root [2019-09-21 09:58:02,678] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-21 09:58:02,678] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-21 09:58:02,679] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-21 09:58:02,878] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [2019-09-21 09:58:02,879] - Script::install(): KBExtra is successfully!
  INFO root [2019-09-21 09:58:03,011] - Dbmgr::initializeEnd: digest(6C1130BA93B38570A1A313CCA3BEAA21)
S_INFO root [2019-09-21 09:58:03,011] - onDBMgrReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2019-09-21 09:58:03,011] - TelnetServer server is running on port 32000
  INFO root [2019-09-21 09:58:03,012] - Dbmgr::initInterfacesHandler: interfaces addr(127.0.0.1:30099), accountType:(interfaces), chargeType:(interfaces).
  WARN root [2019-09-21 09:58:03,262] - DBUtil::initialize: db(default) password is not encrypted!
please use password(rsa):
308F9125B6F2A4887331A4B04EDB09059A66C4C041E8EF2DAD7BFBFD8BA35BE246C1A133E6C634CFDE068F5252EFEC322E7B90B963F4E951C8F3FAA732654B8DB493F4B39DA341AB861A2A8B09D45E6221E7B69806B8D8291FDD34721EAB4141641CD9322C96C9C487A550F2C9BFD89658ABF341D4B98EEEECA9A9C9D86C0A8E
 DEBUG root [2019-09-21 09:58:03,286] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 09:58:03,480] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
  INFO root [2019-09-21 09:58:03,480] - DBUtil::createInterface[0x1afeddc]: interface=default, dbtype=mysql, ip=localhost, port=3306, currdatabase=kbe, username=kbe, connected=yes.

  INFO root [2019-09-21 09:58:03,480] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-21 09:58:03,480] - ThreadPool::createThreadPool: successfully(5), newThreadCount=5, normalMaxThreadCount=5, maxThreadCount=5
 DEBUG root [2019-09-21 09:58:03,488] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 09:58:03,504] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 09:58:03,511] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 09:58:03,515] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 09:58:03,530] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 09:58:03,614] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 09:58:03,614] - DBThread::onStart(): 0x1da1908!
  INFO root [2019-09-21 09:58:03,615] - DBInterfaceMysql::query: successfully!
 DEBUG root [2019-09-21 09:58:03,750] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 09:58:03,750] - DBThread::onStart(): 0x1da1990!
 DEBUG root [2019-09-21 09:58:03,850] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 09:58:03,850] - DBThread::onStart(): 0x1da1330!
  INFO root [2019-09-21 09:58:03,933] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-21 09:58:03,934] - DBTask::process(): took 0.32 seconds
sql:(CREATE TABLE IF NOT EXISTS kbe_accountinfos (`accountName` varchar(128) not null, PRIMARY KEY idKey (`accountName`),`password` varchar(255) not null,`bindata` blob,`email` varchar(191) not null, UNIQUE KEY `email` (`email`),`entityDBID` bigint(20) unsigned not null DEFAULT 0, UNIQUE KEY `entityDBID` (`entityDBID`),`flags` int unsigned not null DEFAULT 0,`deadline` bigint(20) not null DEFAULT 0,`regtime` bigint(20) not null DEFAULT 0,`lasttime` bigint(20) not null DEFAULT 0,`numlogin` int unsigned not null DEFAULT 0)ENGINE=InnoDB)
 DEBUG root [2019-09-21 09:58:04,066] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 09:58:04,066] - DBThread::onStart(): 0x1da1220!
 DEBUG root [2019-09-21 09:58:04,182] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 09:58:04,183] - DBThread::onStart(): 0x1da1088!
  WARN root [2019-09-21 09:58:04,184] - DBTask::process(): took 0.43 seconds
sql:(show index from tbl_Account)
  INFO root [2019-09-21 09:58:04,240] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 09:58:04,258] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 09:58:04,258] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 09:58:04,259] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 09:58:04,259] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 09:58:04,259] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-21 09:58:04,260] - DBTask::process(): took 0.32 seconds
sql:(show index from tbl_Room)
  INFO root [2019-09-21 09:58:04,316] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-21 09:58:04,317] - DBTask::process(): took 0.47 seconds
sql:(select distinct(serverGroupID) from kbe_entitylog)
  WARN root [2019-09-21 09:58:04,407] - d:\kbengine\kbe\src\server\dbmgr\dbmgr.cpp:306: Profile scriptCall took 1.40 seconds
  INFO root [2019-09-21 09:58:04,840] - ---- dbmgr is running ----
 DEBUG root [2019-09-21 09:58:04,841] - Components::process(): Request for the process of identity...
  INFO root [2019-09-21 09:58:10,968] - ServerApp::onRegisterNewApp: uid:41181, username:error_name, componentType:interfaces, componentID:3000, globalorderID=-1, grouporderID=-1, intaddr:10.15.26.195, intport:30099, extaddr:nonsupport, extport:0,  from 127.0.0.1:30099/0/0/0.
  INFO root [2019-09-21 09:58:10,968] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-21 09:58:10,969] - Components::addComponent[interfaces], uid=41181, componentID=3000, globalorderid=1, grouporderid=1, totalcount=1
  WARN root [2019-09-21 09:58:10,969] - TCPPacketReceiver::processPendingEvents(10.15.26.195:52090): Throwing REASON_GENERAL_NETWORK - WSAECONNABORTED
  INFO root [2019-09-21 09:58:11,971] - Components::findComponents: find logger(1)...
  INFO root [2019-09-21 09:58:12,060] - Components::findComponents: found logger, addr:10.15.26.195:52074
  INFO root [2019-09-21 09:58:12,060] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-21 09:58:12,061] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  WARN root [2019-09-21 09:58:12,062] - Forwarding logs to logger[10.15.26.195:52074/0/0/0]...
 ERROR root [2019-09-21 09:59:51,505] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=476209508281, lastReceivedTime=476006455905, inactivityExceptionPeriod=599470137)
  INFO root [2019-09-21 09:59:51,512] - The following logs sent to logger failed:
  WARN root [2019-09-21 09:59:51,512] - ==> [2019-09-21 09:59:51 500] TCPPacketReceiver::processPendingEvents(10.15.26.195:52074): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-21 09:59:51,512] - ==> [2019-09-21 09:59:51 505] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=476209508281, lastReceivedTime=476006455905, inactivityExceptionPeriod=599470137)
  INFO root [2019-09-21 10:01:45,577] - 
  INFO root [2019-09-21 10:01:45,578] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-21 10:01:45,579] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-21 10:01:45,579] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-21 10:01:45,579] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-21 10:01:45,593] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.26.195 ( 10.15.26.195:52159 )
  INFO root [2019-09-21 10:01:45,593] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.26.195:52159, SOMAXCONN=5.
  INFO root [2019-09-21 10:01:45,593] - server-configs:
  INFO root [2019-09-21 10:01:45,593] - 	internalTcpAddr : 10.15.26.195:52159
  INFO root [2019-09-21 10:01:45,593] - 	componentID : 4000
  WARN root [2019-09-21 10:01:45,594] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-21 10:01:47,098] - ---- dbmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: 11178A1C367680938C8E416B64844ACD. Config: Debug X86. Built: 09:55:16 May  8 2019. AppID: 4000. UID: 41181. PID: 12456 ----
  INFO root [2019-09-21 10:01:47,099] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-21 10:01:47,099] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-21 10:01:47,100] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-21 10:01:47,294] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [2019-09-21 10:01:47,294] - Script::install(): KBExtra is successfully!
  INFO root [2019-09-21 10:01:47,424] - Dbmgr::initializeEnd: digest(6C1130BA93B38570A1A313CCA3BEAA21)
S_INFO root [2019-09-21 10:01:47,425] - onDBMgrReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2019-09-21 10:01:47,425] - TelnetServer server is running on port 32000
  INFO root [2019-09-21 10:01:47,425] - Dbmgr::initInterfacesHandler: interfaces addr(127.0.0.1:30099), accountType:(interfaces), chargeType:(interfaces).
  WARN root [2019-09-21 10:01:47,643] - DBUtil::initialize: db(default) password is not encrypted!
please use password(rsa):
9D413CFC35923B2A8EA693EECCDA9110FB204ACABE2231FE8E60B67152FFCAED5BADA1F95FFC57C9851BF7139D1631A6C598FB675BB5FA6C5F1B3522F825986BBC631A325852031CCBB1BE2A9311DA0CBED1D897312F1C1B4DB60DB877A2FE6AA4194687FD7A9C7B719D14198648B9E4C124E61FEAA21A606284ED419BBB98D0
 DEBUG root [2019-09-21 10:01:47,647] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 10:01:47,758] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
  INFO root [2019-09-21 10:01:47,759] - DBUtil::createInterface[0x1fef2c]: interface=default, dbtype=mysql, ip=localhost, port=3306, currdatabase=kbe, username=kbe, connected=yes.

  INFO root [2019-09-21 10:01:47,759] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-21 10:01:47,759] - ThreadPool::createThreadPool: successfully(5), newThreadCount=5, normalMaxThreadCount=5, maxThreadCount=5
 DEBUG root [2019-09-21 10:01:47,759] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 10:01:47,760] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 10:01:47,760] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 10:01:47,760] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 10:01:47,761] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 10:01:47,877] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
  INFO root [2019-09-21 10:01:47,877] - DBInterfaceMysql::query: successfully!
 DEBUG root [2019-09-21 10:01:47,877] - DBThread::onStart(): 0x1910be8!
 DEBUG root [2019-09-21 10:01:47,934] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 10:01:47,934] - DBThread::onStart(): 0x1910c70!
 DEBUG root [2019-09-21 10:01:48,034] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 10:01:48,035] - DBThread::onStart(): 0x1910500!
 DEBUG root [2019-09-21 10:01:48,084] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 10:01:48,084] - DBThread::onStart(): 0x19107a8!
  INFO root [2019-09-21 10:01:48,134] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 10:01:48,151] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 10:01:48,151] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-21 10:01:48,152] - DBTask::process(): took 0.22 seconds
sql:(CREATE TABLE IF NOT EXISTS kbe_accountinfos (`accountName` varchar(128) not null, PRIMARY KEY idKey (`accountName`),`password` varchar(255) not null,`bindata` blob,`email` varchar(191) not null, UNIQUE KEY `email` (`email`),`entityDBID` bigint(20) unsigned not null DEFAULT 0, UNIQUE KEY `entityDBID` (`entityDBID`),`flags` int unsigned not null DEFAULT 0,`deadline` bigint(20) not null DEFAULT 0,`regtime` bigint(20) not null DEFAULT 0,`lasttime` bigint(20) not null DEFAULT 0,`numlogin` int unsigned not null DEFAULT 0)ENGINE=InnoDB)
  INFO root [2019-09-21 10:01:48,243] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-21 10:01:48,244] - DBTask::process(): took 0.37 seconds
sql:(select distinct(serverGroupID) from kbe_entitylog)
  WARN root [2019-09-21 10:01:48,269] - DBTask::process(): took 0.23 seconds
sql:(insert into kbe_serverlog (heartbeatTime, serverGroupID) values(1569031308,41181) ON DUPLICATE KEY UPDATE heartbeatTime=1569031308)
 DEBUG root [2019-09-21 10:01:48,270] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 10:01:48,270] - DBThread::onStart(): 0x1910cf8!
  WARN root [2019-09-21 10:01:48,346] - DBTask::process(): took 0.26 seconds
sql:(show index from tbl_Account)
  INFO root [2019-09-21 10:01:48,359] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 10:01:48,360] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 10:01:48,361] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 10:01:48,361] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-21 10:01:48,362] - DBTask::process(): took 0.21 seconds
sql:(show index from tbl_Room)
  WARN root [2019-09-21 10:01:48,404] - d:\kbengine\kbe\src\server\dbmgr\dbmgr.cpp:306: Profile scriptCall took 0.98 seconds
  INFO root [2019-09-21 10:01:48,825] - ---- dbmgr is running ----
 DEBUG root [2019-09-21 10:01:48,825] - Components::process(): Request for the process of identity...
  INFO root [2019-09-21 10:01:54,948] - ServerApp::onRegisterNewApp: uid:41181, username:error_name, componentType:interfaces, componentID:3000, globalorderID=-1, grouporderID=-1, intaddr:10.15.26.195, intport:30099, extaddr:nonsupport, extport:0,  from 127.0.0.1:30099/0/0/0.
  INFO root [2019-09-21 10:01:54,948] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-21 10:01:54,948] - Components::addComponent[interfaces], uid=41181, componentID=3000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-21 10:01:55,964] - Components::findComponents: find logger(1)...
  INFO root [2019-09-21 10:01:55,987] - Components::findComponents: found logger, addr:10.15.26.195:52158
  INFO root [2019-09-21 10:01:55,988] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-21 10:01:55,988] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  WARN root [2019-09-21 10:01:55,990] - Forwarding logs to logger[10.15.26.195:52158/0/0/0]...
 ERROR root [2019-09-21 10:05:33,656] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=479631030957, lastReceivedTime=479450320255, inactivityExceptionPeriod=599491240)
  INFO root [2019-09-21 10:05:33,661] - The following logs sent to logger failed:
  WARN root [2019-09-21 10:05:33,662] - ==> [2019-09-21 10:05:33 651] TCPPacketReceiver::processPendingEvents(10.15.26.195:52158): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-21 10:05:33,662] - ==> [2019-09-21 10:05:33 658] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=479631030957, lastReceivedTime=479450320255, inactivityExceptionPeriod=599491240)
  INFO root [2019-09-21 10:06:38,523] - 
  INFO root [2019-09-21 10:06:38,528] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-21 10:06:38,529] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-21 10:06:38,529] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-21 10:06:38,529] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-21 10:06:38,542] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.26.195 ( 10.15.26.195:52223 )
  INFO root [2019-09-21 10:06:38,542] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.26.195:52223, SOMAXCONN=5.
  INFO root [2019-09-21 10:06:38,543] - server-configs:
  INFO root [2019-09-21 10:06:38,543] - 	internalTcpAddr : 10.15.26.195:52223
  INFO root [2019-09-21 10:06:38,543] - 	componentID : 4000
  WARN root [2019-09-21 10:06:38,554] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-21 10:06:40,059] - ---- dbmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: 11178A1C367680938C8E416B64844ACD. Config: Debug X86. Built: 09:55:16 May  8 2019. AppID: 4000. UID: 41181. PID: 13976 ----
  INFO root [2019-09-21 10:06:40,060] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-21 10:06:40,060] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-21 10:06:40,060] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-21 10:06:40,267] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [2019-09-21 10:06:40,267] - Script::install(): KBExtra is successfully!
  INFO root [2019-09-21 10:06:40,411] - Dbmgr::initializeEnd: digest(6C1130BA93B38570A1A313CCA3BEAA21)
S_INFO root [2019-09-21 10:06:40,411] - onDBMgrReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2019-09-21 10:06:40,412] - TelnetServer server is running on port 32000
  INFO root [2019-09-21 10:06:40,413] - Dbmgr::initInterfacesHandler: interfaces addr(127.0.0.1:30099), accountType:(interfaces), chargeType:(interfaces).
  WARN root [2019-09-21 10:06:40,649] - DBUtil::initialize: db(default) password is not encrypted!
please use password(rsa):
66894950630A4E5FB9C5A9D0FA404E50E02A9D4AC0C6652AFEDDA65815F22C472C59DB969A3477193DCE6EA8F49F597D2D396DEA154CC8BD7B42F8E5867ABB5846A3BD364087B1D42B9136D21F3B68465F47C3A314378AD3AC3B785E9BBA4EB3DB930DABD05D02884BEF10C0BA1BC0F67BB1D95A627588C05338FB863B3BEA11
 DEBUG root [2019-09-21 10:06:40,653] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 10:06:40,851] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
  INFO root [2019-09-21 10:06:40,852] - DBUtil::createInterface[0x1afeeb4]: interface=default, dbtype=mysql, ip=localhost, port=3306, currdatabase=kbe, username=kbe, connected=yes.

  INFO root [2019-09-21 10:06:40,852] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-21 10:06:40,852] - ThreadPool::createThreadPool: successfully(5), newThreadCount=5, normalMaxThreadCount=5, maxThreadCount=5
 DEBUG root [2019-09-21 10:06:40,853] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 10:06:40,853] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 10:06:40,853] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 10:06:40,854] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 10:06:40,855] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 10:06:40,997] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 10:06:40,997] - DBThread::onStart(): 0x1cdf2d0!
  INFO root [2019-09-21 10:06:40,999] - DBInterfaceMysql::query: successfully!
 DEBUG root [2019-09-21 10:06:41,087] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 10:06:41,087] - DBThread::onStart(): 0x1cdf468!
 DEBUG root [2019-09-21 10:06:41,144] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 10:06:41,145] - DBThread::onStart(): 0x1cdf248!
 DEBUG root [2019-09-21 10:06:41,245] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 10:06:41,245] - DBThread::onStart(): 0x1cdf578!
  INFO root [2019-09-21 10:06:41,294] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 10:06:41,311] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 10:06:41,312] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-21 10:06:41,312] - DBTask::process(): took 0.22 seconds
sql:(CREATE TABLE IF NOT EXISTS kbe_accountinfos (`accountName` varchar(128) not null, PRIMARY KEY idKey (`accountName`),`password` varchar(255) not null,`bindata` blob,`email` varchar(191) not null, UNIQUE KEY `email` (`email`),`entityDBID` bigint(20) unsigned not null DEFAULT 0, UNIQUE KEY `entityDBID` (`entityDBID`),`flags` int unsigned not null DEFAULT 0,`deadline` bigint(20) not null DEFAULT 0,`regtime` bigint(20) not null DEFAULT 0,`lasttime` bigint(20) not null DEFAULT 0,`numlogin` int unsigned not null DEFAULT 0)ENGINE=InnoDB)
  INFO root [2019-09-21 10:06:41,313] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 10:06:41,313] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 10:06:41,314] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-21 10:06:41,314] - DBTask::process(): took 0.31 seconds
sql:(delete from kbe_email_verification where logtime<1569028001 and type=3)
  WARN root [2019-09-21 10:06:41,363] - DBTask::process(): took 0.22 seconds
sql:(insert into kbe_serverlog (heartbeatTime, serverGroupID) values(1569031601,41181) ON DUPLICATE KEY UPDATE heartbeatTime=1569031601)
 DEBUG root [2019-09-21 10:06:41,394] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 10:06:41,394] - DBThread::onStart(): 0x1cdf4f0!
  WARN root [2019-09-21 10:06:41,455] - DBTask::process(): took 0.21 seconds
sql:(show index from tbl_Account)
  INFO root [2019-09-21 10:06:41,486] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 10:06:41,619] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-21 10:06:41,620] - DBTask::process(): took 0.22 seconds
sql:(select distinct(serverGroupID) from kbe_entitylog)
  WARN root [2019-09-21 10:06:41,637] - d:\kbengine\kbe\src\server\dbmgr\dbmgr.cpp:306: Profile scriptCall took 1.23 seconds
  INFO root [2019-09-21 10:06:42,101] - ---- dbmgr is running ----
 DEBUG root [2019-09-21 10:06:42,102] - Components::process(): Request for the process of identity...
  INFO root [2019-09-21 10:06:48,246] - ServerApp::onRegisterNewApp: uid:41181, username:error_name, componentType:interfaces, componentID:3000, globalorderID=-1, grouporderID=-1, intaddr:10.15.26.195, intport:30099, extaddr:nonsupport, extport:0,  from 127.0.0.1:30099/0/0/0.
  INFO root [2019-09-21 10:06:48,246] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-21 10:06:48,247] - Components::addComponent[interfaces], uid=41181, componentID=3000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-21 10:06:49,250] - Components::findComponents: find logger(1)...
  INFO root [2019-09-21 10:06:49,335] - Components::findComponents: found logger, addr:10.15.26.195:52222
  INFO root [2019-09-21 10:06:49,335] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-21 10:06:49,335] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  WARN root [2019-09-21 10:06:49,337] - Forwarding logs to logger[10.15.26.195:52222/0/0/0]...
 ERROR root [2019-09-21 10:27:24,720] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=492741663323, lastReceivedTime=492579517175, inactivityExceptionPeriod=599492071)
  INFO root [2019-09-21 10:27:24,725] - The following logs sent to logger failed:
  WARN root [2019-09-21 10:27:24,726] - ==> [2019-09-21 10:27:24 719] TCPPacketReceiver::processPendingEvents(10.15.26.195:52222): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-21 10:27:24,726] - ==> [2019-09-21 10:27:24 720] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=492741663323, lastReceivedTime=492579517175, inactivityExceptionPeriod=599492071)
  INFO root [2019-09-21 10:52:47,013] - 
  INFO root [2019-09-21 10:52:47,022] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-21 10:52:47,023] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-21 10:52:47,023] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-21 10:52:47,024] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-21 10:52:47,055] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.26.195 ( 10.15.26.195:52804 )
  INFO root [2019-09-21 10:52:47,055] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.26.195:52804, SOMAXCONN=5.
  INFO root [2019-09-21 10:52:47,055] - server-configs:
  INFO root [2019-09-21 10:52:47,055] - 	internalTcpAddr : 10.15.26.195:52804
  INFO root [2019-09-21 10:52:47,056] - 	componentID : 4000
  WARN root [2019-09-21 10:52:47,065] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  WARN root [2019-09-21 10:52:47,486] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20674, errorno=10048
  INFO root [2019-09-21 10:52:47,490] - ---- dbmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: 11178A1C367680938C8E416B64844ACD. Config: Debug X86. Built: 09:55:16 May  8 2019. AppID: 4000. UID: 41181. PID: 3600 ----
  INFO root [2019-09-21 10:52:47,491] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-21 10:52:47,491] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-21 10:52:47,491] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-21 10:52:47,749] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [2019-09-21 10:52:47,749] - Script::install(): KBExtra is successfully!
  INFO root [2019-09-21 10:52:48,031] - Dbmgr::initializeEnd: digest(6C1130BA93B38570A1A313CCA3BEAA21)
S_INFO root [2019-09-21 10:52:48,032] - onDBMgrReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2019-09-21 10:52:48,032] - TelnetServer server is running on port 32000
  INFO root [2019-09-21 10:52:48,042] - Dbmgr::initInterfacesHandler: interfaces addr(127.0.0.1:30099), accountType:(interfaces), chargeType:(interfaces).
  WARN root [2019-09-21 10:52:48,385] - DBUtil::initialize: db(default) password is not encrypted!
please use password(rsa):
7DABD727332D5A4E9600CC3FF933208547F128A3C86CC417A15E0031E5C07B533FCC75B57677454DE33A39623E64AA35E157ED9422AFB038ACD9C1325AD0DC12027936C43A8038D265427C07F98F4E8F06D6C64D2029B07689B970A258515B3F75CEC8C7203560C9561F495D73FFC7377554641B35D2A9C4EC38BA3AED73F094
 DEBUG root [2019-09-21 10:52:48,388] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 10:52:48,549] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
  INFO root [2019-09-21 10:52:48,550] - DBUtil::createInterface[0x17aec48]: interface=default, dbtype=mysql, ip=localhost, port=3306, currdatabase=kbe, username=kbe, connected=yes.

  INFO root [2019-09-21 10:52:48,550] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-21 10:52:48,550] - ThreadPool::createThreadPool: successfully(5), newThreadCount=5, normalMaxThreadCount=5, maxThreadCount=5
 DEBUG root [2019-09-21 10:52:48,555] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 10:52:48,560] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 10:52:48,560] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 10:52:48,569] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 10:52:48,575] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 10:52:48,656] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 10:52:48,656] - DBThread::onStart(): 0x1c0e400!
  INFO root [2019-09-21 10:52:48,784] - DBInterfaceMysql::query: successfully!
 DEBUG root [2019-09-21 10:52:48,845] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 10:52:48,846] - DBThread::onStart(): 0x1c0d9e8!
 DEBUG root [2019-09-21 10:52:48,929] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 10:52:48,930] - DBThread::onStart(): 0x1c0e510!
  INFO root [2019-09-21 10:52:49,054] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 10:52:49,054] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-21 10:52:49,055] - DBTask::process(): took 0.27 seconds
sql:(CREATE TABLE IF NOT EXISTS kbe_accountinfos (`accountName` varchar(128) not null, PRIMARY KEY idKey (`accountName`),`password` varchar(255) not null,`bindata` blob,`email` varchar(191) not null, UNIQUE KEY `email` (`email`),`entityDBID` bigint(20) unsigned not null DEFAULT 0, UNIQUE KEY `entityDBID` (`entityDBID`),`flags` int unsigned not null DEFAULT 0,`deadline` bigint(20) not null DEFAULT 0,`regtime` bigint(20) not null DEFAULT 0,`lasttime` bigint(20) not null DEFAULT 0,`numlogin` int unsigned not null DEFAULT 0)ENGINE=InnoDB)
 DEBUG root [2019-09-21 10:52:49,130] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 10:52:49,131] - DBThread::onStart(): 0x1c0e488!
  WARN root [2019-09-21 10:52:49,263] - DBTask::process(): took 0.33 seconds
sql:(insert into kbe_serverlog (heartbeatTime, serverGroupID) values(1569034369,41181) ON DUPLICATE KEY UPDATE heartbeatTime=1569034369)
 DEBUG root [2019-09-21 10:52:49,323] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 10:52:49,324] - DBThread::onStart(): 0x1c0d8d8!
  INFO root [2019-09-21 10:52:49,403] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 10:52:49,405] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-21 10:52:49,406] - DBTask::process(): took 0.28 seconds
sql:(select distinct(serverGroupID) from kbe_entitylog)
  INFO root [2019-09-21 10:52:49,421] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-21 10:52:49,463] - DBTask::process(): took 0.62 seconds
sql:(show index from tbl_Account)
  WARN root [2019-09-21 10:52:49,463] - DBTask::process(): took 0.41 seconds
sql:(show index from tbl_Room)
  WARN root [2019-09-21 10:52:49,505] - DBTask::process(): took 0.24 seconds
sql:(show index from tbl_Hall)
  INFO root [2019-09-21 10:52:49,514] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 10:52:49,515] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 10:52:49,515] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-21 10:52:49,532] - d:\kbengine\kbe\src\server\dbmgr\dbmgr.cpp:306: Profile scriptCall took 1.50 seconds
  INFO root [2019-09-21 10:52:49,993] - ---- dbmgr is running ----
 DEBUG root [2019-09-21 10:52:49,994] - Components::process(): Request for the process of identity...
  INFO root [2019-09-21 10:52:56,132] - ServerApp::onRegisterNewApp: uid:41181, username:error_name, componentType:interfaces, componentID:3000, globalorderID=-1, grouporderID=-1, intaddr:10.15.26.195, intport:30099, extaddr:nonsupport, extport:0,  from 127.0.0.1:30099/0/0/0.
  INFO root [2019-09-21 10:52:56,133] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-21 10:52:56,133] - Components::addComponent[interfaces], uid=41181, componentID=3000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-21 10:52:57,140] - Components::findComponents: find logger(1)...
  INFO root [2019-09-21 10:52:57,241] - Components::findComponents: found logger, addr:10.15.26.195:52800
  INFO root [2019-09-21 10:52:57,242] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-21 10:52:57,242] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  WARN root [2019-09-21 10:52:57,244] - Forwarding logs to logger[10.15.26.195:52800/0/0/0]...
 ERROR root [2019-09-21 10:54:19,520] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=508889670946, lastReceivedTime=508864548351, inactivityExceptionPeriod=599464062)
  INFO root [2019-09-21 10:54:19,526] - The following logs sent to logger failed:
  WARN root [2019-09-21 10:54:19,526] - ==> [2019-09-21 10:54:19 515] TCPPacketReceiver::processPendingEvents(10.15.26.195:52800): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-21 10:54:19,526] - ==> [2019-09-21 10:54:19 522] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=508889670946, lastReceivedTime=508864548351, inactivityExceptionPeriod=599464062)
  INFO root [2019-09-21 10:54:21,466] - 
  INFO root [2019-09-21 10:54:21,466] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-21 10:54:21,468] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-21 10:54:21,468] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-21 10:54:21,468] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-21 10:54:21,528] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.26.195 ( 10.15.26.195:52882 )
  INFO root [2019-09-21 10:54:21,528] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.26.195:52882, SOMAXCONN=5.
  INFO root [2019-09-21 10:54:21,528] - server-configs:
  INFO root [2019-09-21 10:54:21,528] - 	internalTcpAddr : 10.15.26.195:52882
  INFO root [2019-09-21 10:54:21,528] - 	componentID : 4000
  WARN root [2019-09-21 10:54:21,537] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-21 10:54:23,050] - ---- dbmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: 11178A1C367680938C8E416B64844ACD. Config: Debug X86. Built: 09:55:16 May  8 2019. AppID: 4000. UID: 41181. PID: 12056 ----
  INFO root [2019-09-21 10:54:23,051] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-21 10:54:23,052] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-21 10:54:23,052] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-21 10:54:23,288] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [2019-09-21 10:54:23,289] - Script::install(): KBExtra is successfully!
  INFO root [2019-09-21 10:54:23,438] - Dbmgr::initializeEnd: digest(6C1130BA93B38570A1A313CCA3BEAA21)
S_INFO root [2019-09-21 10:54:23,438] - onDBMgrReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2019-09-21 10:54:23,439] - TelnetServer server is running on port 32000
  INFO root [2019-09-21 10:54:23,439] - Dbmgr::initInterfacesHandler: interfaces addr(127.0.0.1:30099), accountType:(interfaces), chargeType:(interfaces).
  WARN root [2019-09-21 10:54:23,687] - DBUtil::initialize: db(default) password is not encrypted!
please use password(rsa):
E2CB195A09D6E218C888F93A291013803FA5E4656CD4DC29660A6A7E12523ADD3D9A34D31231EDEB1C859E3EC861C9B897289DA90FD5BD800C384AFEE3E5E4AF960494129FF0DA57815887B5796BD9C7B44728FFC8007DD2BACD02ECC0AB90F469EF0BE0F89ED029CE4B14741BEC9735405CD8384FB4125E29FC3EA14E465CC6
 DEBUG root [2019-09-21 10:54:23,692] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 10:54:23,875] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
  INFO root [2019-09-21 10:54:23,875] - DBUtil::createInterface[0x1aff064]: interface=default, dbtype=mysql, ip=localhost, port=3306, currdatabase=kbe, username=kbe, connected=yes.

  INFO root [2019-09-21 10:54:23,875] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-21 10:54:23,876] - ThreadPool::createThreadPool: successfully(5), newThreadCount=5, normalMaxThreadCount=5, maxThreadCount=5
 DEBUG root [2019-09-21 10:54:23,876] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 10:54:23,877] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 10:54:23,877] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 10:54:23,877] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 10:54:23,879] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 10:54:24,009] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 10:54:24,009] - DBThread::onStart(): 0x1c4eaa8!
 DEBUG root [2019-09-21 10:54:24,072] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 10:54:24,072] - DBThread::onStart(): 0x1c4e4d0!
 DEBUG root [2019-09-21 10:54:24,130] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 10:54:24,130] - DBThread::onStart(): 0x1c4e118!
 DEBUG root [2019-09-21 10:54:24,188] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 10:54:24,188] - DBThread::onStart(): 0x1c4e448!
  INFO root [2019-09-21 10:54:24,190] - DBInterfaceMysql::query: successfully!
 DEBUG root [2019-09-21 10:54:24,280] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 10:54:24,280] - DBThread::onStart(): 0x1c4df80!
  INFO root [2019-09-21 10:54:24,355] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 10:54:24,387] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 10:54:24,388] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 10:54:24,388] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 10:54:24,389] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 10:54:24,389] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 10:54:24,390] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-21 10:54:24,572] - DBTask::process(): took 0.38 seconds
sql:(insert into kbe_serverlog (heartbeatTime, serverGroupID) values(1569034464,41181) ON DUPLICATE KEY UPDATE heartbeatTime=1569034464)
  INFO root [2019-09-21 10:54:24,605] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-21 10:54:24,606] - DBTask::process(): took 0.41 seconds
sql:(select distinct(serverGroupID) from kbe_entitylog)
  WARN root [2019-09-21 10:54:24,621] - d:\kbengine\kbe\src\server\dbmgr\dbmgr.cpp:306: Profile scriptCall took 1.18 seconds
  INFO root [2019-09-21 10:54:25,082] - ---- dbmgr is running ----
 DEBUG root [2019-09-21 10:54:25,082] - Components::process(): Request for the process of identity...
  INFO root [2019-09-21 10:54:31,228] - ServerApp::onRegisterNewApp: uid:41181, username:error_name, componentType:interfaces, componentID:3000, globalorderID=-1, grouporderID=-1, intaddr:10.15.26.195, intport:30099, extaddr:nonsupport, extport:0,  from 127.0.0.1:30099/0/0/0.
  INFO root [2019-09-21 10:54:31,229] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-21 10:54:31,229] - Components::addComponent[interfaces], uid=41181, componentID=3000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-21 10:54:32,238] - Components::findComponents: find logger(1)...
  INFO root [2019-09-21 10:54:32,341] - Components::findComponents: found logger, addr:10.15.26.195:52880
  INFO root [2019-09-21 10:54:32,341] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-21 10:54:32,341] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  WARN root [2019-09-21 10:54:32,342] - Forwarding logs to logger[10.15.26.195:52880/0/0/0]...
 ERROR root [2019-09-21 11:09:42,242] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=518116882318, lastReceivedTime=517908115928, inactivityExceptionPeriod=599485996)
  INFO root [2019-09-21 11:09:42,246] - The following logs sent to logger failed:
  WARN root [2019-09-21 11:09:42,247] - ==> [2019-09-21 11:09:42 240] TCPPacketReceiver::processPendingEvents(10.15.26.195:52880): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-21 11:09:42,247] - ==> [2019-09-21 11:09:42 242] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=518116882318, lastReceivedTime=517908115928, inactivityExceptionPeriod=599485996)
  INFO root [2019-09-21 11:10:44,778] - 
  INFO root [2019-09-21 11:10:44,779] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-21 11:10:44,780] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-21 11:10:44,780] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-21 11:10:44,781] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-21 11:10:44,792] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.26.195 ( 10.15.26.195:53003 )
  INFO root [2019-09-21 11:10:44,792] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.26.195:53003, SOMAXCONN=5.
  INFO root [2019-09-21 11:10:44,792] - server-configs:
  INFO root [2019-09-21 11:10:44,793] - 	internalTcpAddr : 10.15.26.195:53003
  INFO root [2019-09-21 11:10:44,793] - 	componentID : 4000
  WARN root [2019-09-21 11:10:44,793] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-21 11:10:46,299] - ---- dbmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: 11178A1C367680938C8E416B64844ACD. Config: Debug X86. Built: 09:55:16 May  8 2019. AppID: 4000. UID: 41181. PID: 10904 ----
  INFO root [2019-09-21 11:10:46,299] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-21 11:10:46,300] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-21 11:10:46,300] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-21 11:10:46,495] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [2019-09-21 11:10:46,496] - Script::install(): KBExtra is successfully!
  INFO root [2019-09-21 11:10:46,629] - Dbmgr::initializeEnd: digest(6C1130BA93B38570A1A313CCA3BEAA21)
S_INFO root [2019-09-21 11:10:46,630] - onDBMgrReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2019-09-21 11:10:46,630] - TelnetServer server is running on port 32000
  INFO root [2019-09-21 11:10:46,631] - Dbmgr::initInterfacesHandler: interfaces addr(127.0.0.1:30099), accountType:(interfaces), chargeType:(interfaces).
  WARN root [2019-09-21 11:10:46,846] - DBUtil::initialize: db(default) password is not encrypted!
please use password(rsa):
17503F9C510D49A9745E7733F57FE3706332ABD4A8CA6C07DD1221B0C2E5B73404DF87CB36F7E4BDADA719A1D9D547120AB9563BD5A86B85852624551D60E5FD48E01DD39AE7B4F42722BCB608A89E8B5E8973E36D4C2F957F5944420C00B70429524BBBB56E6BC9D6C35EC94365628E81ACBBE035CED6D30A5DF0CD2FCC7657
 DEBUG root [2019-09-21 11:10:46,851] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 11:10:47,083] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
  INFO root [2019-09-21 11:10:47,084] - DBUtil::createInterface[0x63f1e4]: interface=default, dbtype=mysql, ip=localhost, port=3306, currdatabase=kbe, username=kbe, connected=yes.

  INFO root [2019-09-21 11:10:47,084] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-21 11:10:47,084] - ThreadPool::createThreadPool: successfully(5), newThreadCount=5, normalMaxThreadCount=5, maxThreadCount=5
 DEBUG root [2019-09-21 11:10:47,085] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 11:10:47,085] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 11:10:47,085] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 11:10:47,085] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 11:10:47,101] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 11:10:47,173] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 11:10:47,173] - DBThread::onStart(): 0x1dfa648!
 DEBUG root [2019-09-21 11:10:47,244] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 11:10:47,244] - DBThread::onStart(): 0x1df9ed8!
 DEBUG root [2019-09-21 11:10:47,302] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 11:10:47,303] - DBThread::onStart(): 0x1dfa6d0!
 DEBUG root [2019-09-21 11:10:47,402] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 11:10:47,403] - DBThread::onStart(): 0x1dfa070!
  INFO root [2019-09-21 11:10:47,405] - DBInterfaceMysql::query: successfully!
 DEBUG root [2019-09-21 11:10:47,494] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 11:10:47,494] - DBThread::onStart(): 0x1dfa5c0!
  INFO root [2019-09-21 11:10:47,585] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 11:10:47,586] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 11:10:47,586] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 11:10:47,586] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 11:10:47,586] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 11:10:47,587] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 11:10:47,588] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 11:10:47,744] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-21 11:10:47,745] - DBTask::process(): took 0.34 seconds
sql:(insert into kbe_serverlog (heartbeatTime, serverGroupID) values(1569035447,41181) ON DUPLICATE KEY UPDATE heartbeatTime=1569035447)
  WARN root [2019-09-21 11:10:47,747] - DBTask::process(): took 0.34 seconds
sql:(select distinct(serverGroupID) from kbe_entitylog)
  WARN root [2019-09-21 11:10:47,778] - d:\kbengine\kbe\src\server\dbmgr\dbmgr.cpp:306: Profile scriptCall took 1.15 seconds
  INFO root [2019-09-21 11:10:48,196] - ---- dbmgr is running ----
 DEBUG root [2019-09-21 11:10:48,197] - Components::process(): Request for the process of identity...
  INFO root [2019-09-21 11:10:54,319] - ServerApp::onRegisterNewApp: uid:41181, username:error_name, componentType:interfaces, componentID:3000, globalorderID=-1, grouporderID=-1, intaddr:10.15.26.195, intport:30099, extaddr:nonsupport, extport:0,  from 127.0.0.1:30099/0/0/0.
  INFO root [2019-09-21 11:10:54,319] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-21 11:10:54,320] - Components::addComponent[interfaces], uid=41181, componentID=3000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-21 11:10:55,329] - Components::findComponents: find logger(1)...
  INFO root [2019-09-21 11:10:55,390] - Components::findComponents: found logger, addr:10.15.26.195:53004
  INFO root [2019-09-21 11:10:55,390] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-21 11:10:55,390] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  WARN root [2019-09-21 11:10:55,391] - Forwarding logs to logger[10.15.26.195:53004/0/0/0]...
 ERROR root [2019-09-21 11:13:18,270] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=520277163187, lastReceivedTime=520242452847, inactivityExceptionPeriod=599493734)
  INFO root [2019-09-21 11:13:18,271] - The following logs sent to logger failed:
  WARN root [2019-09-21 11:13:18,272] - ==> [2019-09-21 11:13:18 262] TCPPacketReceiver::processPendingEvents(10.15.26.195:53004): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-21 11:13:18,272] - ==> [2019-09-21 11:13:18 270] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=520277163187, lastReceivedTime=520242452847, inactivityExceptionPeriod=599493734)
  INFO root [2019-09-21 11:13:20,028] - 
  INFO root [2019-09-21 11:13:20,028] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-21 11:13:20,030] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-21 11:13:20,030] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-21 11:13:20,030] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-21 11:13:20,041] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.26.195 ( 10.15.26.195:53063 )
  INFO root [2019-09-21 11:13:20,042] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.26.195:53063, SOMAXCONN=5.
  INFO root [2019-09-21 11:13:20,042] - server-configs:
  INFO root [2019-09-21 11:13:20,042] - 	internalTcpAddr : 10.15.26.195:53063
  INFO root [2019-09-21 11:13:20,042] - 	componentID : 4000
  WARN root [2019-09-21 11:13:20,042] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-21 11:13:21,548] - ---- dbmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: 11178A1C367680938C8E416B64844ACD. Config: Debug X86. Built: 09:55:16 May  8 2019. AppID: 4000. UID: 41181. PID: 12904 ----
  INFO root [2019-09-21 11:13:21,548] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-21 11:13:21,549] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-21 11:13:21,549] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-21 11:13:21,759] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [2019-09-21 11:13:21,759] - Script::install(): KBExtra is successfully!
  INFO root [2019-09-21 11:13:21,898] - Dbmgr::initializeEnd: digest(6C1130BA93B38570A1A313CCA3BEAA21)
S_INFO root [2019-09-21 11:13:21,899] - onDBMgrReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2019-09-21 11:13:21,899] - TelnetServer server is running on port 32000
  INFO root [2019-09-21 11:13:21,900] - Dbmgr::initInterfacesHandler: interfaces addr(127.0.0.1:30099), accountType:(interfaces), chargeType:(interfaces).
  WARN root [2019-09-21 11:13:22,136] - DBUtil::initialize: db(default) password is not encrypted!
please use password(rsa):
39189867B445D2BD894C31D4CEE32785EAA36EA4E90E2D1626D0D18931FBF1BC5867C6A55171ECC08242989A7E37549BE59FE7814B4E4BCEA39A80D0544F1F46FEE62DA77CA2885E617309274E8B9C2EFE1B6FB9078459AFC2C208BA23EE3D660B8D906005DDA37AB71C679BD66D837BB0E26D104993403C9579E1CE4DEEC207
 DEBUG root [2019-09-21 11:13:22,143] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 11:13:22,292] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
  INFO root [2019-09-21 11:13:22,292] - DBUtil::createInterface[0x53f18c]: interface=default, dbtype=mysql, ip=localhost, port=3306, currdatabase=kbe, username=kbe, connected=yes.

  INFO root [2019-09-21 11:13:22,292] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-21 11:13:22,293] - ThreadPool::createThreadPool: successfully(5), newThreadCount=5, normalMaxThreadCount=5, maxThreadCount=5
 DEBUG root [2019-09-21 11:13:22,297] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 11:13:22,314] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 11:13:22,314] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 11:13:22,314] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 11:13:22,324] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 11:13:22,426] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 11:13:22,427] - DBThread::onStart(): 0x1782a60!
  INFO root [2019-09-21 11:13:22,428] - DBInterfaceMysql::query: successfully!
 DEBUG root [2019-09-21 11:13:22,537] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 11:13:22,537] - DBThread::onStart(): 0x1782d08!
 DEBUG root [2019-09-21 11:13:22,595] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 11:13:22,596] - DBThread::onStart(): 0x1782d90!
  INFO root [2019-09-21 11:13:22,653] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-21 11:13:22,654] - DBTask::process(): took 0.22 seconds
sql:(CREATE TABLE IF NOT EXISTS kbe_accountinfos (`accountName` varchar(128) not null, PRIMARY KEY idKey (`accountName`),`password` varchar(255) not null,`bindata` blob,`email` varchar(191) not null, UNIQUE KEY `email` (`email`),`entityDBID` bigint(20) unsigned not null DEFAULT 0, UNIQUE KEY `entityDBID` (`entityDBID`),`flags` int unsigned not null DEFAULT 0,`deadline` bigint(20) not null DEFAULT 0,`regtime` bigint(20) not null DEFAULT 0,`lasttime` bigint(20) not null DEFAULT 0,`numlogin` int unsigned not null DEFAULT 0)ENGINE=InnoDB)
  INFO root [2019-09-21 11:13:22,669] - DBInterfaceMysql::query: successfully!
 DEBUG root [2019-09-21 11:13:22,712] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 11:13:22,713] - DBThread::onStart(): 0x1782f28!
  WARN root [2019-09-21 11:13:22,770] - DBTask::process(): took 0.23 seconds
sql:(insert into kbe_serverlog (heartbeatTime, serverGroupID) values(1569035602,41181) ON DUPLICATE KEY UPDATE heartbeatTime=1569035602)
 DEBUG root [2019-09-21 11:13:22,880] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 11:13:22,880] - DBThread::onStart(): 0x17830c0!
  WARN root [2019-09-21 11:13:23,007] - DBTask::process(): took 0.35 seconds
sql:(show index from tbl_Room)
  INFO root [2019-09-21 11:13:23,037] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 11:13:23,037] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 11:13:23,038] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 11:13:23,039] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 11:13:23,039] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-21 11:13:23,039] - DBTask::process(): took 0.27 seconds
sql:(show index from tbl_Hall)
  WARN root [2019-09-21 11:13:23,040] - DBTask::process(): took 0.45 seconds
sql:(show index from tbl_Account)
  INFO root [2019-09-21 11:13:23,262] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-21 11:13:23,264] - DBTask::process(): took 0.55 seconds
sql:(select distinct(serverGroupID) from kbe_entitylog)
  WARN root [2019-09-21 11:13:23,284] - d:\kbengine\kbe\src\server\dbmgr\dbmgr.cpp:306: Profile scriptCall took 1.39 seconds
  INFO root [2019-09-21 11:13:23,717] - ---- dbmgr is running ----
 DEBUG root [2019-09-21 11:13:23,717] - Components::process(): Request for the process of identity...
  INFO root [2019-09-21 11:13:29,843] - ServerApp::onRegisterNewApp: uid:41181, username:error_name, componentType:interfaces, componentID:3000, globalorderID=-1, grouporderID=-1, intaddr:10.15.26.195, intport:30099, extaddr:nonsupport, extport:0,  from 127.0.0.1:30099/0/0/0.
  INFO root [2019-09-21 11:13:29,844] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-21 11:13:29,844] - Components::addComponent[interfaces], uid=41181, componentID=3000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-21 11:13:30,859] - Components::findComponents: find logger(1)...
  INFO root [2019-09-21 11:13:30,922] - Components::findComponents: found logger, addr:10.15.26.195:53069
  INFO root [2019-09-21 11:13:30,922] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-21 11:13:30,923] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  WARN root [2019-09-21 11:13:30,932] - Forwarding logs to logger[10.15.26.195:53069/0/0/0]...
 ERROR root [2019-09-21 11:16:26,649] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=522160956500, lastReceivedTime=522096295161, inactivityExceptionPeriod=599489577)
  INFO root [2019-09-21 11:16:26,655] - The following logs sent to logger failed:
  WARN root [2019-09-21 11:16:26,656] - ==> [2019-09-21 11:16:26 647] TCPPacketReceiver::processPendingEvents(10.15.26.195:53069): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-21 11:16:26,657] - ==> [2019-09-21 11:16:26 650] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=522160956500, lastReceivedTime=522096295161, inactivityExceptionPeriod=599489577)
  INFO root [2019-09-21 11:16:31,279] - 
  INFO root [2019-09-21 11:16:31,280] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-21 11:16:31,281] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-21 11:16:31,281] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-21 11:16:31,281] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-21 11:16:31,340] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.26.195 ( 10.15.26.195:53122 )
  INFO root [2019-09-21 11:16:31,340] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.26.195:53122, SOMAXCONN=5.
  INFO root [2019-09-21 11:16:31,340] - server-configs:
  INFO root [2019-09-21 11:16:31,340] - 	internalTcpAddr : 10.15.26.195:53122
  INFO root [2019-09-21 11:16:31,341] - 	componentID : 4000
  WARN root [2019-09-21 11:16:31,345] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  WARN root [2019-09-21 11:16:31,789] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20720, errorno=10048
  INFO root [2019-09-21 11:16:31,793] - ---- dbmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: 11178A1C367680938C8E416B64844ACD. Config: Debug X86. Built: 09:55:16 May  8 2019. AppID: 4000. UID: 41181. PID: 6360 ----
  INFO root [2019-09-21 11:16:31,793] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-21 11:16:31,794] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-21 11:16:31,794] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-21 11:16:32,005] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [2019-09-21 11:16:32,006] - Script::install(): KBExtra is successfully!
  INFO root [2019-09-21 11:16:32,155] - Dbmgr::initializeEnd: digest(6C1130BA93B38570A1A313CCA3BEAA21)
S_INFO root [2019-09-21 11:16:32,155] - onDBMgrReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2019-09-21 11:16:32,155] - TelnetServer server is running on port 32000
  INFO root [2019-09-21 11:16:32,156] - Dbmgr::initInterfacesHandler: interfaces addr(127.0.0.1:30099), accountType:(interfaces), chargeType:(interfaces).
  WARN root [2019-09-21 11:16:32,403] - DBUtil::initialize: db(default) password is not encrypted!
please use password(rsa):
660DBC07A1E703628229F55EBC1FF428BB907D91DC7F6C143462CC25293EEFAEBFE8D76F82A97B23B705665A31EC3760E02191222AD878AD85DE09E0BFC47F4DF715DB0F565415A907832E9C805A5061F9B282326987066A016167C341DAB610C439806B762822C22AB33D5AA03F0AE13494C483CF2D94E132446D50D7CB7CE5
 DEBUG root [2019-09-21 11:16:32,407] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 11:16:32,547] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
  INFO root [2019-09-21 11:16:32,547] - DBUtil::createInterface[0x1aff2b4]: interface=default, dbtype=mysql, ip=localhost, port=3306, currdatabase=kbe, username=kbe, connected=yes.

  INFO root [2019-09-21 11:16:32,547] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-21 11:16:32,548] - ThreadPool::createThreadPool: successfully(5), newThreadCount=5, normalMaxThreadCount=5, maxThreadCount=5
 DEBUG root [2019-09-21 11:16:32,548] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 11:16:32,548] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 11:16:32,556] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 11:16:32,564] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 11:16:32,569] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 11:16:32,671] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 11:16:32,671] - DBThread::onStart(): 0x1e732b8!
 DEBUG root [2019-09-21 11:16:32,738] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 11:16:32,739] - DBThread::onStart(): 0x1e72ce0!
 DEBUG root [2019-09-21 11:16:32,813] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 11:16:32,813] - DBThread::onStart(): 0x1e72a38!
  INFO root [2019-09-21 11:16:32,910] - DBInterfaceMysql::query: successfully!
 DEBUG root [2019-09-21 11:16:32,980] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 11:16:32,980] - DBThread::onStart(): 0x1e725f8!
 DEBUG root [2019-09-21 11:16:33,041] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 11:16:33,041] - DBThread::onStart(): 0x1e72bd0!
  INFO root [2019-09-21 11:16:33,105] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 11:16:33,147] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-21 11:16:33,148] - DBTask::process(): took 0.24 seconds
sql:(CREATE TABLE IF NOT EXISTS kbe_accountinfos (`accountName` varchar(128) not null, PRIMARY KEY idKey (`accountName`),`password` varchar(255) not null,`bindata` blob,`email` varchar(191) not null, UNIQUE KEY `email` (`email`),`entityDBID` bigint(20) unsigned not null DEFAULT 0, UNIQUE KEY `entityDBID` (`entityDBID`),`flags` int unsigned not null DEFAULT 0,`deadline` bigint(20) not null DEFAULT 0,`regtime` bigint(20) not null DEFAULT 0,`lasttime` bigint(20) not null DEFAULT 0,`numlogin` int unsigned not null DEFAULT 0)ENGINE=InnoDB)
  INFO root [2019-09-21 11:16:33,151] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 11:16:33,151] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 11:16:33,223] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 11:16:33,224] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 11:16:33,224] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-21 11:16:33,224] - DBTask::process(): took 0.24 seconds
sql:(delete from kbe_email_verification where logtime<1569032193 and type=3)
  WARN root [2019-09-21 11:16:33,305] - DBTask::process(): took 0.26 seconds
sql:(show index from tbl_Account)
  INFO root [2019-09-21 11:16:33,315] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-21 11:16:33,316] - DBTask::process(): took 0.41 seconds
sql:(select distinct(serverGroupID) from kbe_entitylog)
  WARN root [2019-09-21 11:16:33,346] - DBTask::process(): took 0.44 seconds
sql:(insert into kbe_serverlog (heartbeatTime, serverGroupID) values(1569035793,41181) ON DUPLICATE KEY UPDATE heartbeatTime=1569035793)
  WARN root [2019-09-21 11:16:33,504] - d:\kbengine\kbe\src\server\dbmgr\dbmgr.cpp:306: Profile scriptCall took 1.35 seconds
  INFO root [2019-09-21 11:16:33,949] - ---- dbmgr is running ----
 DEBUG root [2019-09-21 11:16:33,950] - Components::process(): Request for the process of identity...
  INFO root [2019-09-21 11:16:40,088] - ServerApp::onRegisterNewApp: uid:41181, username:error_name, componentType:interfaces, componentID:3000, globalorderID=-1, grouporderID=-1, intaddr:10.15.26.195, intport:30099, extaddr:nonsupport, extport:0,  from 127.0.0.1:30099/0/0/0.
  INFO root [2019-09-21 11:16:40,089] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-21 11:16:40,089] - Components::addComponent[interfaces], uid=41181, componentID=3000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-21 11:16:41,095] - Components::findComponents: find logger(1)...
  INFO root [2019-09-21 11:16:41,160] - Components::findComponents: found logger, addr:10.15.26.195:53118
  INFO root [2019-09-21 11:16:41,160] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-21 11:16:41,161] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  WARN root [2019-09-21 11:16:41,162] - Forwarding logs to logger[10.15.26.195:53118/0/0/0]...
 ERROR root [2019-09-21 11:17:01,824] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=522512706393, lastReceivedTime=522506267641, inactivityExceptionPeriod=599487915)
  INFO root [2019-09-21 11:17:01,831] - The following logs sent to logger failed:
  WARN root [2019-09-21 11:17:01,831] - ==> [2019-09-21 11:17:01 824] TCPPacketReceiver::processPendingEvents(10.15.26.195:53118): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-21 11:17:01,831] - ==> [2019-09-21 11:17:01 825] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=522512706393, lastReceivedTime=522506267641, inactivityExceptionPeriod=599487915)
  INFO root [2019-09-21 11:17:03,615] - 
  INFO root [2019-09-21 11:17:03,632] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-21 11:17:03,634] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-21 11:17:03,635] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-21 11:17:03,635] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-21 11:17:03,661] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.26.195 ( 10.15.26.195:53158 )
  INFO root [2019-09-21 11:17:03,662] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.26.195:53158, SOMAXCONN=5.
  INFO root [2019-09-21 11:17:03,662] - server-configs:
  INFO root [2019-09-21 11:17:03,662] - 	internalTcpAddr : 10.15.26.195:53158
  INFO root [2019-09-21 11:17:03,662] - 	componentID : 4000
  WARN root [2019-09-21 11:17:03,663] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-21 11:17:05,168] - ---- dbmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: 11178A1C367680938C8E416B64844ACD. Config: Debug X86. Built: 09:55:16 May  8 2019. AppID: 4000. UID: 41181. PID: 8172 ----
  INFO root [2019-09-21 11:17:05,168] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-21 11:17:05,169] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-21 11:17:05,169] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-21 11:17:05,465] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [2019-09-21 11:17:05,465] - Script::install(): KBExtra is successfully!
  INFO root [2019-09-21 11:17:05,740] - Dbmgr::initializeEnd: digest(6C1130BA93B38570A1A313CCA3BEAA21)
S_INFO root [2019-09-21 11:17:05,741] - onDBMgrReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2019-09-21 11:17:05,741] - TelnetServer server is running on port 32000
  INFO root [2019-09-21 11:17:05,743] - Dbmgr::initInterfacesHandler: interfaces addr(127.0.0.1:30099), accountType:(interfaces), chargeType:(interfaces).
  WARN root [2019-09-21 11:17:05,969] - DBUtil::initialize: db(default) password is not encrypted!
please use password(rsa):
7EADA141CDD36D3E1AAFD3907824E5098FC9D3B8EF0B5C294E18C5AE1234BFD89508AA4373974899212E1832A2EB5112110B40F65FBE8BED678DD800A97F2269C515812D0DDD416F9BC346913206CC5119903FEBAECB9FDDDC15A306BE1D76234F60E1F5C4E20610B18E3981631C52BBDB9BCA7B35FD235FCE89BBA108F4E2F7
 DEBUG root [2019-09-21 11:17:05,973] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 11:17:06,112] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
  INFO root [2019-09-21 11:17:06,112] - DBUtil::createInterface[0x1aff384]: interface=default, dbtype=mysql, ip=localhost, port=3306, currdatabase=kbe, username=kbe, connected=yes.

  INFO root [2019-09-21 11:17:06,113] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-21 11:17:06,113] - ThreadPool::createThreadPool: successfully(5), newThreadCount=5, normalMaxThreadCount=5, maxThreadCount=5
 DEBUG root [2019-09-21 11:17:06,115] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 11:17:06,115] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 11:17:06,115] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 11:17:06,115] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 11:17:06,116] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 11:17:06,264] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 11:17:06,264] - DBThread::onStart(): 0x1d5e6f0!
 DEBUG root [2019-09-21 11:17:06,422] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 11:17:06,423] - DBThread::onStart(): 0x1d5da30!
  INFO root [2019-09-21 11:17:06,424] - DBInterfaceMysql::query: successfully!
 DEBUG root [2019-09-21 11:17:06,598] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 11:17:06,598] - DBThread::onStart(): 0x1d5dab8!
 DEBUG root [2019-09-21 11:17:06,715] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 11:17:06,715] - DBThread::onStart(): 0x1d5dbc8!
  INFO root [2019-09-21 11:17:06,796] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 11:17:06,822] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 11:17:06,822] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-21 11:17:06,823] - DBTask::process(): took 0.40 seconds
sql:(CREATE TABLE IF NOT EXISTS kbe_accountinfos (`accountName` varchar(128) not null, PRIMARY KEY idKey (`accountName`),`password` varchar(255) not null,`bindata` blob,`email` varchar(191) not null, UNIQUE KEY `email` (`email`),`entityDBID` bigint(20) unsigned not null DEFAULT 0, UNIQUE KEY `entityDBID` (`entityDBID`),`flags` int unsigned not null DEFAULT 0,`deadline` bigint(20) not null DEFAULT 0,`regtime` bigint(20) not null DEFAULT 0,`lasttime` bigint(20) not null DEFAULT 0,`numlogin` int unsigned not null DEFAULT 0)ENGINE=InnoDB)
  INFO root [2019-09-21 11:17:06,823] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 11:17:06,823] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 11:17:06,824] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-21 11:17:06,824] - DBTask::process(): took 0.23 seconds
sql:(delete from kbe_email_verification where logtime<1569032226 and type=3)
  WARN root [2019-09-21 11:17:06,905] - DBTask::process(): took 0.48 seconds
sql:(insert into kbe_serverlog (heartbeatTime, serverGroupID) values(1569035826,41181) ON DUPLICATE KEY UPDATE heartbeatTime=1569035826)
 DEBUG root [2019-09-21 11:17:06,922] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 11:17:06,923] - DBThread::onStart(): 0x1d5db40!
  INFO root [2019-09-21 11:17:07,063] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 11:17:07,064] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-21 11:17:07,065] - DBTask::process(): took 0.24 seconds
sql:(show index from tbl_Hall)
  WARN root [2019-09-21 11:17:07,065] - DBTask::process(): took 0.24 seconds
sql:(show index from tbl_Room)
  WARN root [2019-09-21 11:17:07,067] - DBTask::process(): took 0.35 seconds
sql:(show index from tbl_Account)
  WARN root [2019-09-21 11:17:07,117] - d:\kbengine\kbe\src\server\dbmgr\dbmgr.cpp:306: Profile scriptCall took 1.38 seconds
  INFO root [2019-09-21 11:17:07,551] - ---- dbmgr is running ----
 DEBUG root [2019-09-21 11:17:07,552] - Components::process(): Request for the process of identity...
  INFO root [2019-09-21 11:17:13,692] - ServerApp::onRegisterNewApp: uid:41181, username:error_name, componentType:interfaces, componentID:3000, globalorderID=-1, grouporderID=-1, intaddr:10.15.26.195, intport:30099, extaddr:nonsupport, extport:0,  from 127.0.0.1:30099/0/0/0.
  INFO root [2019-09-21 11:17:13,693] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-21 11:17:13,693] - Components::addComponent[interfaces], uid=41181, componentID=3000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-21 11:17:14,701] - Components::findComponents: find logger(1)...
  INFO root [2019-09-21 11:17:14,758] - Components::findComponents: found logger, addr:10.15.26.195:53153
  INFO root [2019-09-21 11:17:14,759] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-21 11:17:14,759] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  WARN root [2019-09-21 11:17:14,760] - Forwarding logs to logger[10.15.26.195:53153/0/0/0]...
 ERROR root [2019-09-21 11:20:54,432] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=524838786759, lastReceivedTime=524629257108, inactivityExceptionPeriod=599493414)
  INFO root [2019-09-21 11:20:54,444] - The following logs sent to logger failed:
  WARN root [2019-09-21 11:20:54,444] - ==> [2019-09-21 11:20:54 428] TCPPacketReceiver::processPendingEvents(10.15.26.195:53153): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-21 11:20:54,445] - ==> [2019-09-21 11:20:54 433] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=524838786759, lastReceivedTime=524629257108, inactivityExceptionPeriod=599493414)
  INFO root [2019-09-21 14:44:23,241] - 
  INFO root [2019-09-21 14:44:23,254] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-21 14:44:23,255] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-21 14:44:23,255] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-21 14:44:23,255] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-21 14:44:23,266] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.26.195 ( 10.15.26.195:54424 )
  INFO root [2019-09-21 14:44:23,266] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.26.195:54424, SOMAXCONN=5.
  INFO root [2019-09-21 14:44:23,266] - server-configs:
  INFO root [2019-09-21 14:44:23,266] - 	internalTcpAddr : 10.15.26.195:54424
  INFO root [2019-09-21 14:44:23,267] - 	componentID : 4000
  WARN root [2019-09-21 14:44:23,267] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-21 14:44:24,771] - ---- dbmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: 11178A1C367680938C8E416B64844ACD. Config: Debug X86. Built: 09:55:16 May  8 2019. AppID: 4000. UID: 41181. PID: 8804 ----
  INFO root [2019-09-21 14:44:24,772] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-21 14:44:24,773] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-21 14:44:24,773] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-21 14:44:24,966] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [2019-09-21 14:44:24,966] - Script::install(): KBExtra is successfully!
  INFO root [2019-09-21 14:44:25,092] - Dbmgr::initializeEnd: digest(D6F52F242106EC594C99DE19E090C909)
S_INFO root [2019-09-21 14:44:25,092] - onDBMgrReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2019-09-21 14:44:25,093] - TelnetServer server is running on port 32000
  INFO root [2019-09-21 14:44:25,095] - Dbmgr::initInterfacesHandler: interfaces addr(127.0.0.1:30099), accountType:(interfaces), chargeType:(interfaces).
  WARN root [2019-09-21 14:44:25,303] - DBUtil::initialize: db(default) password is not encrypted!
please use password(rsa):
62603BC9ED39D0C9C42F5B588CEC5F4F5BFDBD62712B2E5DEE1500C09F44E25E0CA9DEC432846A10D652DC5CEAA9953A81725BE15F3824AFE5E17E759E7BE355655B7C1E47024B9329E799F59BDF427D3C51317BBF4362EE4E5054BA857F55F9D6D6A00730B40EC2DC97723A1145C4459FA05D464492348BB3F6BE4E50BCE71E
 DEBUG root [2019-09-21 14:44:25,367] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 14:44:25,539] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
  INFO root [2019-09-21 14:44:25,540] - DBUtil::createInterface[0x19df364]: interface=default, dbtype=mysql, ip=localhost, port=3306, currdatabase=kbe, username=kbe, connected=yes.

  INFO root [2019-09-21 14:44:25,540] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-21 14:44:25,540] - ThreadPool::createThreadPool: successfully(5), newThreadCount=5, normalMaxThreadCount=5, maxThreadCount=5
 DEBUG root [2019-09-21 14:44:25,581] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 14:44:25,597] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 14:44:25,622] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 14:44:25,638] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 14:44:25,639] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 14:44:25,687] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 14:44:25,687] - DBThread::onStart(): 0x1f52530!
  INFO root [2019-09-21 14:44:25,689] - DBInterfaceMysql::query: successfully!
 DEBUG root [2019-09-21 14:44:25,813] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 14:44:25,813] - DBThread::onStart(): 0x1f527d8!
 DEBUG root [2019-09-21 14:44:26,005] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 14:44:26,005] - DBThread::onStart(): 0x1f51dc0!
  INFO root [2019-09-21 14:44:26,105] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-21 14:44:26,105] - DBTask::process(): took 0.42 seconds
sql:(CREATE TABLE IF NOT EXISTS kbe_accountinfos (`accountName` varchar(128) not null, PRIMARY KEY idKey (`accountName`),`password` varchar(255) not null,`bindata` blob,`email` varchar(191) not null, UNIQUE KEY `email` (`email`),`entityDBID` bigint(20) unsigned not null DEFAULT 0, UNIQUE KEY `entityDBID` (`entityDBID`),`flags` int unsigned not null DEFAULT 0,`deadline` bigint(20) not null DEFAULT 0,`regtime` bigint(20) not null DEFAULT 0,`lasttime` bigint(20) not null DEFAULT 0,`numlogin` int unsigned not null DEFAULT 0)ENGINE=InnoDB)
  INFO root [2019-09-21 14:44:26,138] - DBInterfaceMysql::query: successfully!
 DEBUG root [2019-09-21 14:44:26,205] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 14:44:26,206] - DBThread::onStart(): 0x1f51fe0!
  INFO root [2019-09-21 14:44:26,295] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-21 14:44:26,296] - DBTask::process(): took 0.48 seconds
sql:(select distinct(serverGroupID) from kbe_entitylog)
 DEBUG root [2019-09-21 14:44:26,330] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 14:44:26,330] - DBThread::onStart(): 0x1f52640!
  INFO root [2019-09-21 14:44:26,420] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 14:44:26,421] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 14:44:26,421] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 14:44:26,422] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-21 14:44:26,422] - DBTask::process(): took 0.42 seconds
sql:(delete from kbe_email_verification where logtime<1569044666 and type=3)
  INFO root [2019-09-21 14:44:26,437] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-21 14:44:26,440] - DBTask::process(): took 0.23 seconds
sql:(show index from tbl_Account)
  WARN root [2019-09-21 14:44:26,723] - DBTask::process(): took 0.39 seconds
sql:(insert into kbe_serverlog (heartbeatTime, serverGroupID) values(1569048266,41181) ON DUPLICATE KEY UPDATE heartbeatTime=1569048266)
  WARN root [2019-09-21 14:44:26,804] - DBTask::process(): took 0.38 seconds
sql:(show index from tbl_Food)
  WARN root [2019-09-21 14:44:26,807] - DBTask::process(): took 0.70 seconds
sql:(show index from tbl_Room)
  WARN root [2019-09-21 14:44:26,829] - DBTask::process(): took 0.39 seconds
sql:(show index from tbl_Smash)
  WARN root [2019-09-21 14:44:26,850] - d:\kbengine\kbe\src\server\dbmgr\dbmgr.cpp:306: Profile scriptCall took 1.76 seconds
  INFO root [2019-09-21 14:44:27,244] - ---- dbmgr is running ----
 DEBUG root [2019-09-21 14:44:27,244] - Components::process(): Request for the process of identity...
  INFO root [2019-09-21 14:44:33,341] - ServerApp::onRegisterNewApp: uid:41181, username:error_name, componentType:interfaces, componentID:3000, globalorderID=-1, grouporderID=-1, intaddr:10.15.26.195, intport:30099, extaddr:nonsupport, extport:0,  from 127.0.0.1:30099/0/0/0.
  INFO root [2019-09-21 14:44:33,341] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-21 14:44:33,341] - Components::addComponent[interfaces], uid=41181, componentID=3000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-21 14:44:34,352] - Components::findComponents: find logger(1)...
  INFO root [2019-09-21 14:44:34,368] - Components::findComponents: found logger, addr:10.15.26.195:54427
  INFO root [2019-09-21 14:44:34,368] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-21 14:44:34,369] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  WARN root [2019-09-21 14:44:34,370] - Forwarding logs to logger[10.15.26.195:54427/0/0/0]...
 ERROR root [2019-09-21 14:54:59,595] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=653385837393, lastReceivedTime=653324853745, inactivityExceptionPeriod=599491623)
  INFO root [2019-09-21 14:54:59,597] - The following logs sent to logger failed:
  WARN root [2019-09-21 14:54:59,598] - ==> [2019-09-21 14:54:59 594] TCPPacketReceiver::processPendingEvents(10.15.26.195:54427): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-21 14:54:59,598] - ==> [2019-09-21 14:54:59 595] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=653385837393, lastReceivedTime=653324853745, inactivityExceptionPeriod=599491623)
  INFO root [2019-09-21 15:02:27,009] - 
  INFO root [2019-09-21 15:02:27,010] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-21 15:02:27,012] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-21 15:02:27,013] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-21 15:02:27,013] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-21 15:02:27,024] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.26.195 ( 10.15.26.195:54610 )
  INFO root [2019-09-21 15:02:27,024] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.26.195:54610, SOMAXCONN=5.
  INFO root [2019-09-21 15:02:27,024] - server-configs:
  INFO root [2019-09-21 15:02:27,024] - 	internalTcpAddr : 10.15.26.195:54610
  INFO root [2019-09-21 15:02:27,025] - 	componentID : 4000
  WARN root [2019-09-21 15:02:27,025] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-21 15:02:28,531] - ---- dbmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: 11178A1C367680938C8E416B64844ACD. Config: Debug X86. Built: 09:55:16 May  8 2019. AppID: 4000. UID: 41181. PID: 11200 ----
  INFO root [2019-09-21 15:02:28,531] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-21 15:02:28,532] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-21 15:02:28,532] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-21 15:02:28,731] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [2019-09-21 15:02:28,731] - Script::install(): KBExtra is successfully!
  INFO root [2019-09-21 15:02:28,866] - Dbmgr::initializeEnd: digest(D6F52F242106EC594C99DE19E090C909)
S_INFO root [2019-09-21 15:02:28,866] - onDBMgrReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2019-09-21 15:02:28,867] - TelnetServer server is running on port 32000
  INFO root [2019-09-21 15:02:28,868] - Dbmgr::initInterfacesHandler: interfaces addr(127.0.0.1:30099), accountType:(interfaces), chargeType:(interfaces).
  WARN root [2019-09-21 15:02:29,107] - DBUtil::initialize: db(default) password is not encrypted!
please use password(rsa):
73D80C5E5ED3554CB9A99047E67D63D60FCE57078684790FA3E28782B0A58CCFB5A82DA5AC7717F4AA5E1B3A52AAA39977EEC1FDB2FB58064316CD0DB48EC06269EE97092DD62A54DDB018552F53492D51091BCBD93DE0690E5E0309848E2240C71A784DEE86D7482502F77948E23A1E5B8A12DFE090EB00B88B9CFDF167E5C7
 DEBUG root [2019-09-21 15:02:29,111] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 15:02:29,321] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
  INFO root [2019-09-21 15:02:29,321] - DBUtil::createInterface[0x19df214]: interface=default, dbtype=mysql, ip=localhost, port=3306, currdatabase=kbe, username=kbe, connected=yes.

  INFO root [2019-09-21 15:02:29,321] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-21 15:02:29,322] - ThreadPool::createThreadPool: successfully(5), newThreadCount=5, normalMaxThreadCount=5, maxThreadCount=5
 DEBUG root [2019-09-21 15:02:29,323] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 15:02:29,326] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 15:02:29,329] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 15:02:29,337] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 15:02:29,339] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 15:02:29,430] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 15:02:29,430] - DBThread::onStart(): 0x2047328!
  INFO root [2019-09-21 15:02:29,432] - DBInterfaceMysql::query: successfully!
 DEBUG root [2019-09-21 15:02:29,544] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 15:02:29,544] - DBThread::onStart(): 0x2046dd8!
 DEBUG root [2019-09-21 15:02:29,637] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 15:02:29,637] - DBThread::onStart(): 0x2046f70!
 DEBUG root [2019-09-21 15:02:29,719] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 15:02:29,720] - DBThread::onStart(): 0x20477f0!
  INFO root [2019-09-21 15:02:29,777] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 15:02:29,811] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 15:02:29,811] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-21 15:02:29,811] - DBTask::process(): took 0.38 seconds
sql:(CREATE TABLE IF NOT EXISTS kbe_accountinfos (`accountName` varchar(128) not null, PRIMARY KEY idKey (`accountName`),`password` varchar(255) not null,`bindata` blob,`email` varchar(191) not null, UNIQUE KEY `email` (`email`),`entityDBID` bigint(20) unsigned not null DEFAULT 0, UNIQUE KEY `entityDBID` (`entityDBID`),`flags` int unsigned not null DEFAULT 0,`deadline` bigint(20) not null DEFAULT 0,`regtime` bigint(20) not null DEFAULT 0,`lasttime` bigint(20) not null DEFAULT 0,`numlogin` int unsigned not null DEFAULT 0)ENGINE=InnoDB)
  WARN root [2019-09-21 15:02:29,928] - DBTask::process(): took 0.38 seconds
sql:(insert into kbe_serverlog (heartbeatTime, serverGroupID) values(1569049349,41181) ON DUPLICATE KEY UPDATE heartbeatTime=1569049349)
  INFO root [2019-09-21 15:02:30,019] - DBInterfaceMysql::query: successfully!
 DEBUG root [2019-09-21 15:02:30,020] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 15:02:30,020] - DBThread::onStart(): 0x2047548!
  WARN root [2019-09-21 15:02:30,020] - DBTask::process(): took 0.38 seconds
sql:(select distinct(serverGroupID) from kbe_entitylog)
  INFO root [2019-09-21 15:02:30,143] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 15:02:30,144] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 15:02:30,144] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 15:02:30,145] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-21 15:02:30,145] - DBTask::process(): took 0.43 seconds
sql:(delete from kbe_email_verification where logtime<1569045750 and type=3)
  WARN root [2019-09-21 15:02:30,146] - DBTask::process(): took 0.33 seconds
sql:(show index from tbl_Room)
  WARN root [2019-09-21 15:02:30,171] - DBTask::process(): took 0.24 seconds
sql:(show index from tbl_Hall)
  WARN root [2019-09-21 15:02:30,222] - d:\kbengine\kbe\src\server\dbmgr\dbmgr.cpp:306: Profile scriptCall took 1.36 seconds
  INFO root [2019-09-21 15:02:30,660] - ---- dbmgr is running ----
 DEBUG root [2019-09-21 15:02:30,660] - Components::process(): Request for the process of identity...
  INFO root [2019-09-21 15:02:36,835] - ServerApp::onRegisterNewApp: uid:41181, username:error_name, componentType:interfaces, componentID:3000, globalorderID=-1, grouporderID=-1, intaddr:10.15.26.195, intport:30099, extaddr:nonsupport, extport:0,  from 127.0.0.1:30099/0/0/0.
  INFO root [2019-09-21 15:02:36,835] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-21 15:02:36,835] - Components::addComponent[interfaces], uid=41181, componentID=3000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-21 15:02:37,846] - Components::findComponents: find logger(1)...
  INFO root [2019-09-21 15:02:37,904] - Components::findComponents: found logger, addr:10.15.26.195:54614
  INFO root [2019-09-21 15:02:37,905] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-21 15:02:37,905] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-21 15:02:37,914] - Components::process(): Found all the components!
  WARN root [2019-09-21 15:02:37,914] - Forwarding logs to logger[10.15.26.195:54614/0/0/0]...
 ERROR root [2019-09-21 15:11:42,115] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=663411043516, lastReceivedTime=663260762972, inactivityExceptionPeriod=599475445)
  INFO root [2019-09-21 15:11:42,117] - The following logs sent to logger failed:
  WARN root [2019-09-21 15:11:42,118] - ==> [2019-09-21 15:11:42 112] TCPPacketReceiver::processPendingEvents(10.15.26.195:54614): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-21 15:11:42,118] - ==> [2019-09-21 15:11:42 116] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=663411043516, lastReceivedTime=663260762972, inactivityExceptionPeriod=599475445)
  INFO root [2019-09-21 15:11:43,693] - 
  INFO root [2019-09-21 15:11:43,693] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-21 15:11:43,694] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-21 15:11:43,695] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-21 15:11:43,695] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-21 15:11:43,705] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.26.195 ( 10.15.26.195:54719 )
  INFO root [2019-09-21 15:11:43,705] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.26.195:54719, SOMAXCONN=5.
  INFO root [2019-09-21 15:11:43,705] - server-configs:
  INFO root [2019-09-21 15:11:43,706] - 	internalTcpAddr : 10.15.26.195:54719
  INFO root [2019-09-21 15:11:43,706] - 	componentID : 4000
  WARN root [2019-09-21 15:11:43,706] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-21 15:11:45,213] - ---- dbmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: 11178A1C367680938C8E416B64844ACD. Config: Debug X86. Built: 09:55:16 May  8 2019. AppID: 4000. UID: 41181. PID: 14584 ----
  INFO root [2019-09-21 15:11:45,215] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-21 15:11:45,216] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-21 15:11:45,216] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-21 15:11:45,414] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [2019-09-21 15:11:45,414] - Script::install(): KBExtra is successfully!
  INFO root [2019-09-21 15:11:45,548] - Dbmgr::initializeEnd: digest(D6F52F242106EC594C99DE19E090C909)
S_INFO root [2019-09-21 15:11:45,549] - onDBMgrReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2019-09-21 15:11:45,549] - TelnetServer server is running on port 32000
  INFO root [2019-09-21 15:11:45,550] - Dbmgr::initInterfacesHandler: interfaces addr(127.0.0.1:30099), accountType:(interfaces), chargeType:(interfaces).
  WARN root [2019-09-21 15:11:45,770] - DBUtil::initialize: db(default) password is not encrypted!
please use password(rsa):
75FD6D1833E41632B425D5FF257E9D9F075CC57873F917E603A264E37D6DFC21985DB938A3187CF65B09800B7B02856C2CF6389DB2134D893F6D14825AF5D0A03E636C50E528FCF5FB4BD80A590B016B7C594C4E0A32F9EE8CD766A4A56B871DF6B6ABA181F258521AAF97CF0A40BFDE530257878A48C40C1AF48FDCC9A046B2
 DEBUG root [2019-09-21 15:11:45,775] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 15:11:45,994] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
  INFO root [2019-09-21 15:11:45,994] - DBUtil::createInterface[0x4fed70]: interface=default, dbtype=mysql, ip=localhost, port=3306, currdatabase=kbe, username=kbe, connected=yes.

  INFO root [2019-09-21 15:11:45,995] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-21 15:11:45,995] - ThreadPool::createThreadPool: successfully(5), newThreadCount=5, normalMaxThreadCount=5, maxThreadCount=5
 DEBUG root [2019-09-21 15:11:46,003] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 15:11:46,006] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 15:11:46,013] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 15:11:46,030] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 15:11:46,035] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 15:11:46,101] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 15:11:46,102] - DBThread::onStart(): 0x5a1000!
 DEBUG root [2019-09-21 15:11:46,189] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 15:11:46,189] - DBThread::onStart(): 0x5a0ef0!
 DEBUG root [2019-09-21 15:11:46,265] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 15:11:46,265] - DBThread::onStart(): 0x5a1110!
  INFO root [2019-09-21 15:11:46,335] - DBInterfaceMysql::query: successfully!
 DEBUG root [2019-09-21 15:11:46,397] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 15:11:46,397] - DBThread::onStart(): 0x3d3ec10!
 DEBUG root [2019-09-21 15:11:46,473] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 15:11:46,473] - DBThread::onStart(): 0x3d3e170!
  INFO root [2019-09-21 15:11:46,531] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 15:11:46,564] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 15:11:46,564] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 15:11:46,564] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-21 15:11:46,565] - DBTask::process(): took 0.23 seconds
sql:(CREATE TABLE IF NOT EXISTS kbe_accountinfos (`accountName` varchar(128) not null, PRIMARY KEY idKey (`accountName`),`password` varchar(255) not null,`bindata` blob,`email` varchar(191) not null, UNIQUE KEY `email` (`email`),`entityDBID` bigint(20) unsigned not null DEFAULT 0, UNIQUE KEY `entityDBID` (`entityDBID`),`flags` int unsigned not null DEFAULT 0,`deadline` bigint(20) not null DEFAULT 0,`regtime` bigint(20) not null DEFAULT 0,`lasttime` bigint(20) not null DEFAULT 0,`numlogin` int unsigned not null DEFAULT 0)ENGINE=InnoDB)
  INFO root [2019-09-21 15:11:46,617] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 15:11:46,617] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 15:11:46,618] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-21 15:11:46,700] - DBTask::process(): took 0.30 seconds
sql:(show index from tbl_Account)
  WARN root [2019-09-21 15:11:46,732] - DBTask::process(): took 0.40 seconds
sql:(insert into kbe_serverlog (heartbeatTime, serverGroupID) values(1569049906,41181) ON DUPLICATE KEY UPDATE heartbeatTime=1569049906)
  INFO root [2019-09-21 15:11:46,881] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-21 15:11:46,882] - DBTask::process(): took 0.55 seconds
sql:(select distinct(serverGroupID) from kbe_entitylog)
  WARN root [2019-09-21 15:11:46,898] - d:\kbengine\kbe\src\server\dbmgr\dbmgr.cpp:306: Profile scriptCall took 1.35 seconds
  INFO root [2019-09-21 15:11:47,342] - ---- dbmgr is running ----
 DEBUG root [2019-09-21 15:11:47,343] - Components::process(): Request for the process of identity...
  INFO root [2019-09-21 15:11:53,469] - ServerApp::onRegisterNewApp: uid:41181, username:error_name, componentType:interfaces, componentID:3000, globalorderID=-1, grouporderID=-1, intaddr:10.15.26.195, intport:30099, extaddr:nonsupport, extport:0,  from 127.0.0.1:30099/0/0/0.
  INFO root [2019-09-21 15:11:53,469] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-21 15:11:53,470] - Components::addComponent[interfaces], uid=41181, componentID=3000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-21 15:11:54,489] - Components::findComponents: find logger(1)...
  INFO root [2019-09-21 15:11:54,538] - Components::findComponents: found logger, addr:10.15.26.195:54721
  INFO root [2019-09-21 15:11:54,540] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-21 15:11:54,540] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  WARN root [2019-09-21 15:11:54,542] - Forwarding logs to logger[10.15.26.195:54721/0/0/0]...
 ERROR root [2019-09-21 15:22:59,162] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=670181515387, lastReceivedTime=670027390662, inactivityExceptionPeriod=599488106)
  INFO root [2019-09-21 15:22:59,165] - The following logs sent to logger failed:
  WARN root [2019-09-21 15:22:59,165] - ==> [2019-09-21 15:22:59 160] TCPPacketReceiver::processPendingEvents(10.15.26.195:54721): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-21 15:22:59,165] - ==> [2019-09-21 15:22:59 163] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=670181515387, lastReceivedTime=670027390662, inactivityExceptionPeriod=599488106)
  INFO root [2019-09-21 15:35:45,397] - 
  INFO root [2019-09-21 15:35:45,398] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-21 15:35:45,399] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-21 15:35:45,399] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-21 15:35:45,399] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-21 15:35:45,409] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.26.195 ( 10.15.26.195:55691 )
  INFO root [2019-09-21 15:35:45,409] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.26.195:55691, SOMAXCONN=5.
  INFO root [2019-09-21 15:35:45,409] - server-configs:
  INFO root [2019-09-21 15:35:45,410] - 	internalTcpAddr : 10.15.26.195:55691
  INFO root [2019-09-21 15:35:45,410] - 	componentID : 4000
  WARN root [2019-09-21 15:35:45,412] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-21 15:35:46,918] - ---- dbmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: 11178A1C367680938C8E416B64844ACD. Config: Debug X86. Built: 09:55:16 May  8 2019. AppID: 4000. UID: 41181. PID: 9828 ----
  INFO root [2019-09-21 15:35:46,918] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-21 15:35:46,919] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-21 15:35:46,919] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-21 15:35:47,110] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [2019-09-21 15:35:47,110] - Script::install(): KBExtra is successfully!
  INFO root [2019-09-21 15:35:47,236] - Dbmgr::initializeEnd: digest(26669E675CE88DCE7B5C5015B9EC024B)
S_INFO root [2019-09-21 15:35:47,237] - onDBMgrReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2019-09-21 15:35:47,237] - TelnetServer server is running on port 32000
  INFO root [2019-09-21 15:35:47,237] - Dbmgr::initInterfacesHandler: interfaces addr(127.0.0.1:30099), accountType:(interfaces), chargeType:(interfaces).
  WARN root [2019-09-21 15:35:47,459] - DBUtil::initialize: db(default) password is not encrypted!
please use password(rsa):
B6479BBF01ADEEB3EB3382B37574F538ED840E6D3CDAB5F81B5BBB3F6265B0CA2D248AC9B1D0D8DE77B6A7C016DDE0AA6489FB93CFFFEDED06BB3F3731DCEBB2D484B5D0E6867E5037496E6176002720F0D12C13815304E7D37F0B8F788E274C5943B580674F4DB92B9A72F6580BD15B0B92ACCB63DF909A836CD0FDB9C85212
 DEBUG root [2019-09-21 15:35:47,463] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 15:35:47,578] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
  INFO root [2019-09-21 15:35:47,578] - DBUtil::createInterface[0x19df388]: interface=default, dbtype=mysql, ip=localhost, port=3306, currdatabase=kbe, username=kbe, connected=yes.

  INFO root [2019-09-21 15:35:47,579] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-21 15:35:47,579] - ThreadPool::createThreadPool: successfully(5), newThreadCount=5, normalMaxThreadCount=5, maxThreadCount=5
 DEBUG root [2019-09-21 15:35:47,579] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 15:35:47,580] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 15:35:47,580] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 15:35:47,580] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 15:35:47,595] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 15:35:47,671] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 15:35:47,672] - DBThread::onStart(): 0x1f77318!
 DEBUG root [2019-09-21 15:35:47,756] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 15:35:47,756] - DBThread::onStart(): 0x1f77b10!
 DEBUG root [2019-09-21 15:35:47,830] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 15:35:47,831] - DBThread::onStart(): 0x1f774b0!
  INFO root [2019-09-21 15:35:47,832] - DBInterfaceMysql::query: successfully!
 DEBUG root [2019-09-21 15:35:47,915] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 15:35:47,915] - DBThread::onStart(): 0x1f77a88!
 DEBUG root [2019-09-21 15:35:48,022] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 15:35:48,022] - DBThread::onStart(): 0x1f77db8!
  INFO root [2019-09-21 15:35:48,136] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 15:35:48,172] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 15:35:48,172] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 15:35:48,172] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-21 15:35:48,173] - DBTask::process(): took 0.26 seconds
sql:(CREATE TABLE IF NOT EXISTS kbe_accountinfos (`accountName` varchar(128) not null, PRIMARY KEY idKey (`accountName`),`password` varchar(255) not null,`bindata` blob,`email` varchar(191) not null, UNIQUE KEY `email` (`email`),`entityDBID` bigint(20) unsigned not null DEFAULT 0, UNIQUE KEY `entityDBID` (`entityDBID`),`flags` int unsigned not null DEFAULT 0,`deadline` bigint(20) not null DEFAULT 0,`regtime` bigint(20) not null DEFAULT 0,`lasttime` bigint(20) not null DEFAULT 0,`numlogin` int unsigned not null DEFAULT 0)ENGINE=InnoDB)
  INFO root [2019-09-21 15:35:48,173] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 15:35:48,173] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 15:35:48,173] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-21 15:35:48,174] - DBTask::process(): took 0.34 seconds
sql:(show index from tbl_Account)
  INFO root [2019-09-21 15:35:48,298] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-21 15:35:48,299] - DBTask::process(): took 0.47 seconds
sql:(select distinct(serverGroupID) from kbe_entitylog)
  WARN root [2019-09-21 15:35:48,322] - DBTask::process(): took 0.49 seconds
sql:(insert into kbe_serverlog (heartbeatTime, serverGroupID) values(1569051348,41181) ON DUPLICATE KEY UPDATE heartbeatTime=1569051348)
  WARN root [2019-09-21 15:35:48,414] - d:\kbengine\kbe\src\server\dbmgr\dbmgr.cpp:306: Profile scriptCall took 1.18 seconds
  INFO root [2019-09-21 15:35:48,830] - ---- dbmgr is running ----
 DEBUG root [2019-09-21 15:35:48,830] - Components::process(): Request for the process of identity...
  WARN root [2019-09-21 15:35:52,248] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20800, errorno=10048
  INFO root [2019-09-21 15:35:55,295] - ServerApp::onRegisterNewApp: uid:41181, username:error_name, componentType:interfaces, componentID:3000, globalorderID=-1, grouporderID=-1, intaddr:10.15.26.195, intport:30099, extaddr:nonsupport, extport:0,  from 127.0.0.1:30099/0/0/0.
  INFO root [2019-09-21 15:35:55,296] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-21 15:35:55,296] - Components::addComponent[interfaces], uid=41181, componentID=3000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-21 15:35:56,301] - Components::findComponents: find logger(1)...
  INFO root [2019-09-21 15:35:56,388] - Components::findComponents: found logger, addr:10.15.26.195:55689
  INFO root [2019-09-21 15:35:56,389] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-21 15:35:56,389] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  WARN root [2019-09-21 15:35:56,390] - Forwarding logs to logger[10.15.26.195:55689/0/0/0]...
 ERROR root [2019-09-21 15:39:58,338] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=680373278693, lastReceivedTime=680243414108, inactivityExceptionPeriod=599472567)
  INFO root [2019-09-21 15:39:58,343] - The following logs sent to logger failed:
  WARN root [2019-09-21 15:39:58,343] - ==> [2019-09-21 15:39:58 334] TCPPacketReceiver::processPendingEvents(10.15.26.195:55689): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-21 15:39:58,344] - ==> [2019-09-21 15:39:58 340] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=680373278693, lastReceivedTime=680243414108, inactivityExceptionPeriod=599472567)
  INFO root [2019-09-21 15:40:00,073] - 
  INFO root [2019-09-21 15:40:00,085] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-21 15:40:00,088] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-21 15:40:00,089] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-21 15:40:00,089] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-21 15:40:00,126] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.26.195 ( 10.15.26.195:55780 )
  INFO root [2019-09-21 15:40:00,126] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.26.195:55780, SOMAXCONN=5.
  INFO root [2019-09-21 15:40:00,127] - server-configs:
  INFO root [2019-09-21 15:40:00,127] - 	internalTcpAddr : 10.15.26.195:55780
  INFO root [2019-09-21 15:40:00,127] - 	componentID : 4000
  WARN root [2019-09-21 15:40:00,142] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-21 15:40:01,649] - ---- dbmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: 11178A1C367680938C8E416B64844ACD. Config: Debug X86. Built: 09:55:16 May  8 2019. AppID: 4000. UID: 41181. PID: 972 ----
  INFO root [2019-09-21 15:40:01,660] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-21 15:40:01,677] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-21 15:40:01,677] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-21 15:40:01,931] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [2019-09-21 15:40:01,932] - Script::install(): KBExtra is successfully!
  INFO root [2019-09-21 15:40:02,083] - Dbmgr::initializeEnd: digest(26669E675CE88DCE7B5C5015B9EC024B)
S_INFO root [2019-09-21 15:40:02,084] - onDBMgrReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2019-09-21 15:40:02,084] - TelnetServer server is running on port 32000
  INFO root [2019-09-21 15:40:02,085] - Dbmgr::initInterfacesHandler: interfaces addr(127.0.0.1:30099), accountType:(interfaces), chargeType:(interfaces).
  WARN root [2019-09-21 15:40:02,315] - DBUtil::initialize: db(default) password is not encrypted!
please use password(rsa):
4A374D796795A7F5F7DEAF187A9A0EB0FE779C70DCBB7987931AB8D8D2BAE09145428411262B29B630FBB4F30B1F4ED9547C17E52E5CDD1D8DFF5C872A43CD1612C4FE49C87705B1F4994892842B048E412C74BBE8AEC490584135A32FF66353D97FDE00539C2CCBFA201CABA352BEEB3D2070FDE3A530D1B9A58FA3B47687EA
 DEBUG root [2019-09-21 15:40:02,318] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 15:40:02,563] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
  INFO root [2019-09-21 15:40:02,563] - DBUtil::createInterface[0x19defa0]: interface=default, dbtype=mysql, ip=localhost, port=3306, currdatabase=kbe, username=kbe, connected=yes.

  INFO root [2019-09-21 15:40:02,563] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-21 15:40:02,564] - ThreadPool::createThreadPool: successfully(5), newThreadCount=5, normalMaxThreadCount=5, maxThreadCount=5
 DEBUG root [2019-09-21 15:40:02,565] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 15:40:02,565] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 15:40:02,565] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 15:40:02,565] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 15:40:02,580] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2019-09-21 15:40:02,688] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 15:40:02,689] - DBThread::onStart(): 0x1e91770!
 DEBUG root [2019-09-21 15:40:02,757] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 15:40:02,757] - DBThread::onStart(): 0x1e90e68!
 DEBUG root [2019-09-21 15:40:02,848] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 15:40:02,849] - DBThread::onStart(): 0x1e91a18!
  INFO root [2019-09-21 15:40:02,850] - DBInterfaceMysql::query: successfully!
 DEBUG root [2019-09-21 15:40:02,941] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 15:40:02,941] - DBThread::onStart(): 0x1e90de0!
 DEBUG root [2019-09-21 15:40:03,032] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2019-09-21 15:40:03,032] - DBThread::onStart(): 0x1e917f8!
  INFO root [2019-09-21 15:40:03,090] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 15:40:03,115] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 15:40:03,115] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 15:40:03,116] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 15:40:03,116] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 15:40:03,116] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-21 15:40:03,117] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-21 15:40:03,117] - DBTask::process(): took 0.27 seconds
sql:(delete from kbe_email_verification where logtime<1569048003 and type=3)
  INFO root [2019-09-21 15:40:03,230] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-21 15:40:03,231] - DBTask::process(): took 0.38 seconds
sql:(select distinct(serverGroupID) from kbe_entitylog)
  WARN root [2019-09-21 15:40:03,281] - DBTask::process(): took 0.43 seconds
sql:(insert into kbe_serverlog (heartbeatTime, serverGroupID) values(1569051603,41181) ON DUPLICATE KEY UPDATE heartbeatTime=1569051603)
  WARN root [2019-09-21 15:40:03,366] - d:\kbengine\kbe\src\server\dbmgr\dbmgr.cpp:306: Profile scriptCall took 1.28 seconds
  INFO root [2019-09-21 15:40:03,886] - ---- dbmgr is running ----
 DEBUG root [2019-09-21 15:40:03,889] - Components::process(): Request for the process of identity...
  INFO root [2019-09-21 15:40:10,058] - ServerApp::onRegisterNewApp: uid:41181, username:error_name, componentType:interfaces, componentID:3000, globalorderID=-1, grouporderID=-1, intaddr:10.15.26.195, intport:30099, extaddr:nonsupport, extport:0,  from 127.0.0.1:30099/0/0/0.
  INFO root [2019-09-21 15:40:10,058] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-21 15:40:10,058] - Components::addComponent[interfaces], uid=41181, componentID=3000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-21 15:40:11,063] - Components::findComponents: find logger(1)...
  INFO root [2019-09-21 15:40:11,078] - Components::findComponents: found logger, addr:10.15.26.195:55781
  INFO root [2019-09-21 15:40:11,078] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-21 15:40:11,078] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-21 15:40:11,083] - Components::process(): Found all the components!
  WARN root [2019-09-21 15:40:11,097] - Forwarding logs to logger[10.15.26.195:55781/0/0/0]...
 ERROR root [2019-09-21 15:55:18,418] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=689574080023, lastReceivedTime=689391131021, inactivityExceptionPeriod=599493862)
  INFO root [2019-09-21 15:55:18,419] - The following logs sent to logger failed:
  WARN root [2019-09-21 15:55:18,419] - ==> [2019-09-21 15:55:18 419] TCPPacketReceiver::processPendingEvents(10.15.26.195:55781): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-21 15:55:18,420] - ==> [2019-09-21 15:55:18 419] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=689574080023, lastReceivedTime=689391131021, inactivityExceptionPeriod=599493862)
